<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.8.25">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


<title>Papers – Nick Eriksson</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
</style>


<script src="site_libs/quarto-nav/quarto-nav.js"></script>
<script src="site_libs/clipboard/clipboard.min.js"></script>
<script src="site_libs/quarto-html/quarto.js" type="module"></script>
<script src="site_libs/quarto-html/tabsets/tabsets.js" type="module"></script>
<script src="site_libs/quarto-html/axe/axe-check.js" type="module"></script>
<script src="site_libs/quarto-html/popper.min.js"></script>
<script src="site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="site_libs/quarto-html/anchor.min.js"></script>
<link href="site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="site_libs/quarto-html/quarto-syntax-highlighting-7b89279ff1a6dce999919e0e67d4d9ec.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="site_libs/bootstrap/bootstrap.min.js"></script>
<link href="site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="site_libs/bootstrap/bootstrap-97d3a2109331f44eb50e14aa3d0258f0.min.css" rel="stylesheet" append-hash="true" id="quarto-bootstrap" data-mode="light">
<script id="quarto-search-options" type="application/json">{
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-text-placeholder": "",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit",
    "search-label": "Search"
  }
}</script>


<link rel="stylesheet" href="styles.css">
</head>

<body class="nav-fixed quarto-light">

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
    <nav class="navbar navbar-expand-lg " data-bs-theme="dark">
      <div class="navbar-container container-fluid">
      <div class="navbar-brand-container mx-auto">
    <a href="./index.html" class="navbar-brand navbar-brand-logo">
    </a>
    <a class="navbar-brand" href="./index.html">
    <span class="navbar-title">Nick Eriksson</span>
    </a>
  </div>
          <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarCollapse" aria-controls="navbarCollapse" role="menu" aria-expanded="false" aria-label="Toggle navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
  <span class="navbar-toggler-icon"></span>
</button>
          <div class="collapse navbar-collapse" id="navbarCollapse">
            <ul class="navbar-nav navbar-nav-scroll me-auto">
  <li class="nav-item">
    <a class="nav-link" href="./index.html"> 
<span class="menu-text">Home</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link active" href="./papers.html" aria-current="page"> 
<span class="menu-text">Papers</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./pdf/eriksson-cv.pdf"> 
<span class="menu-text">CV</span></a>
  </li>  
</ul>
            <ul class="navbar-nav navbar-nav-scroll ms-auto">
  <li class="nav-item">
    <a class="nav-link" href="mailto:nick.eriksson@gmail.com"> 
<span class="menu-text">nick.eriksson@gmail.com</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="http://scholar.google.com/citations?user=5kf1vNwAAAAJ"> <i class="bi bi-google" role="img">
</i> 
<span class="menu-text">Google Scholar</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="https://www.linkedin.com/in/nick-eriksson-1571549/"> <i class="bi bi-linkedin" role="img">
</i> 
<span class="menu-text">LinkedIn</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="https://github.com/nkeriks"> <i class="bi bi-github" role="img">
</i> 
<span class="menu-text">Github</span></a>
  </li>  
</ul>
          </div> <!-- /navcollapse -->
            <div class="quarto-navbar-tools">
</div>
      </div> <!-- /container-fluid -->
    </nav>
</header>
<!-- content -->
<div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article page-navbar">
<!-- sidebar -->
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar zindex-bottom">
        
    </div>
<!-- main -->
<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title">Papers</h1>
</div>



<div class="quarto-title-meta">

    
  
    
  </div>
  


</header>


<div class="cell">
<div class="cell-output-display">
<div class="paper-list">
<div>
<div class="grid">
<div class="g-col-2">
<h4 class="anchored">62</h4>
<img src="img/mr_alcohol.png" class="thumbnail">
</div>
<div class="g-col-1"></div>
<div class="g-col-9">
<p></p>
<p class="paper">
<strong>A phenome-wide association and Mendelian randomisation study of alcohol use variants in a diverse cohort comprising over 3 million individuals</strong>
</p>
<p class="authors paper">Mariela V Jennings, José Jaime Martínez-Magaña, Natasia S Courchesne-Krak, Renata B Cupertino, Laura Vilar-Ribó, Sevim B Bianchi, Alexander S Hatoum, Elizabeth G Atkinson, Paola Giusti-Rodriguez, Janitza L Montalvo-Ortiz, Joel Gelernter, María Soler Artigas, Stella Aslibekyan, Adam Auton, Elizabeth Babalola, Robert K Bell, Jessica Bielenberg, Katarzyna Bryc, Emily Bullis, Daniella Coker, Gabriel Cuellar Partida, Devika Dhamija, Sayantan Das, Sarah L Elson, Nicholas Eriksson, Teresa Filshtein, Alison Fitch, Kipper Fletez-Brant, Pierre Fontanillas, Will Freyman, Julie M Granka, Karl Heilbron, Alejandro Hernandez, Barry Hicks, David A Hinds, Ethan M Jewett, Yunxuan Jiang, Katelyn Kukar, Alan Kwong, Keng-Han Lin, Bianca A Llamas, Maya Lowe, Jey C McCreight, Matthew H McIntyre, Steven J Micheletti, Meghan E Moreno, Priyanka Nandakumar, Dominique T Nguyen, Elizabeth S Noblin, Jared O'Connell, Aaron A Petrakovitz, G David Poznik, Alexandra Reynoso, Morgan Schumacher, Anjali J Shastri, Janie F Shelton, Jingchunzi Shi, Suyash Shringarpure, Qiaojuan Jane Su, Susana A Tat, Christophe Toukam Tchakouté, Vinh Tran, Joyce Y Tung, Xin Wang, Wei Wang, Catherine H Weldon, Peter Wilton, Corinna D Wong, Howard J Edenberg, Abraham A Palmer, Sandra Sanchez-Roige</p>
<p class="paper"><i>eBioMedicine</i> (2024)</p>
<p class="paper">
<strong>Summary: </strong>
Mendelian randomization identifies many causal effects of alcohol consumption
</p>
<p class="paper">
<strong>Links: </strong>
<a href="https://www.thelancet.com/journals/ebiom/article/PIIS2352-3964(24)00121-X/fulltext">eBioMedicine</a>
</p>
</div>
</div>
<hr>
</div>
<div>
<div class="grid">
<div class="g-col-2">
<h4 class="anchored">61</h4>
<img src="img/mussel.png" class="thumbnail">
</div>
<div class="g-col-1"></div>
<div class="g-col-9">
<p></p>
<p class="paper">
<strong>MUSSEL: enhanced Bayesian polygenic risk prediction leveraging information across multiple ancestry groups</strong>
</p>
<p class="authors paper">Jin Jin, Jianan Zhan, Jingning Zhang, Ruzhang Zhao, Jared O’Connell, Yunxuan Jiang, Stella Aslibekyan, Adam Auton, Elizabeth Babalola, Robert K Bell, Jessica Bielenberg, Katarzyna Bryc, Emily Bullis, Daniella Coker, Gabriel Cuellar Partida, Devika Dhamija, Sayantan Das, Sarah L Elson, Nicholas Eriksson, Teresa Filshtein, Alison Fitch, Kipper Fletez-Brant, Pierre Fontanillas, Will Freyman, Julie M Granka, Karl Heilbron, Alejandro Hernandez, Barry Hicks, David A Hinds, Ethan M Jewett, Katelyn Kukar, Alan Kwong, Keng-Han Lin, Bianca A Llamas, Maya Lowe, Jey C McCreight, Matthew H McIntyre, Steven J Micheletti, Meghan E Moreno, Priyanka Nandakumar, Dominique T Nguyen, Elizabeth S Noblin, Aaron A Petrakovitz, G David Poznik, Alexandra Reynoso, Morgan Schumacher, Anjali J Shastri, Janie F Shelton, Jingchunzi Shi, Suyash Shringarpure, Qiaojuan Jane Su, Susana A Tat, Christophe Toukam Tchakouté, Vinh Tran, Joyce Y Tung, Xin Wang, Wei Wang, Catherine H Weldon, Peter Wilton, Corinna D Wong, Steven Buyske, Christopher Gignoux, Christopher Haiman, Eimear E Kenny, Charles Kooperberg, Kari North, Bertram L Koelsch, Genevieve Wojcik, Haoyu Zhang, Nilanjan Chatterjee</p>
<p class="paper"><i>Cell Genomics</i> (2024)</p>
<p class="paper">
<strong>Summary: </strong>
Cross ancestry polygenic risk prediction
</p>
<p class="paper">
<strong>Links: </strong>
<a href="https://www.cell.com/cell-genomics/fulltext/S2666-979X(24)00095-8">Cell Genomics</a>
</p>
</div>
</div>
<hr>
</div>
<div>
<div class="grid">
<div class="g-col-2">
<h4 class="anchored">60</h4>
<img src="img/incident.png" class="thumbnail">
</div>
<div class="g-col-1"></div>
<div class="g-col-9">
<p></p>
<p class="paper">
<strong>Prospective analysis of incident disease among individuals of diverse ancestries using genetic and conventional risk factors</strong>
</p>
<p class="authors paper">Wei Wang, Nicholas Eriksson, Matthew McIntyre, Rafaela Bagur Quetglas, Bertram L Koelsch, David A Hinds, Stella Aslibekyan, Adam Auton, Michael V Holmes, Suyash S Shringarpure</p>
<p class="paper"><i>medRxiv</i> (2023)</p>
<p class="paper">
<strong>Summary: </strong>
Genetics and incident disease risk
</p>
<p class="paper">
<strong>Links: </strong>
<a href="https://www.medrxiv.org/node/718872.full">medRxiv</a>
</p>
</div>
</div>
<hr>
</div>
<div>
<div class="grid">
<div class="g-col-2">
<h4 class="anchored">59</h4>
<img src="img/hand.png" class="thumbnail">
</div>
<div class="g-col-1"></div>
<div class="g-col-9">
<p></p>
<p class="paper">
<strong>Genome-wide association study identifies 48 common genetic variants associated with handedness</strong>
</p>
<p class="authors paper">Gabriel Cuellar-Partida, Joyce Y. Tung, N. Eriksson, Eva Albrecht, Fazil Aliev, Ole A. Andreassen, Inês Barroso, Jacques S. Beckmann, Marco P. Boks, Dorret I. Boomsma, Heather A. Boyd, Monique M. B. Breteler, Harry Campbell, Daniel I. Chasman, Lynn F. Cherkas, Gail Davies, Eco J. C. de Geus, Ian J. Deary, Panos Deloukas, Danielle M. Dick, David L. Duffy, Johan G. Eriksson, Tõnu Esko, Bjarke Feenstra, Frank Geller, Christian Gieger, Ina Giegling, Scott D. Gordon, Jiali Han, Thomas F. Hansen, Annette M. Hartmann, Caroline Hayward, Kauko Heikkilä, Andrew A. Hicks, Joel N. Hirschhorn, Jouke-Jan Hottenga, Jennifer E. Huffman, Liang-Dar Hwang, M. Arfan Ikram, Jaakko Kaprio, John P. Kemp, Kay-Tee Khaw, Norman Klopp, Bettina Konte, Zoltan Kutalik, Jari Lahti, Xin Li, Ruth J. F. Loos, Michelle Luciano, Sigurdur H. Magnusson, Massimo Mangino, Pedro Marques-Vidal, Nicholas G. Martin, Wendy L. McArdle, Mark I. McCarthy, Carolina Medina-Gomez, Mads Melbye, Scott A. Melville, Andres Metspalu, Lili Milani, Vincent Mooser, Mari Nelis, Dale R. Nyholt, Kevin S. O’Connell, Roel A. Ophoff, Cameron Palmer, Aarno Palotie, Teemu Palviainen, Guillaume Pare, Lavinia Paternoster, Leena Peltonen, Brenda W. J. H. Penninx, Ozren Polasek, Peter P. Pramstaller, Inga Prokopenko, Katri Raikkonen, Samuli Ripatti, Fernando Rivadeneira, Igor Rudan, Dan Rujescu, Johannes H. Smit, George Davey Smith, Jordan W. Smoller, Nicole Soranzo, Tim D. Spector, Beate St Pourcain, John M. Starr, Hreinn Stefánsson, Stacy Steinberg, Maris Teder-Laving, Gudmar Thorleifsson, Kári Stefánsson, Nicholas J. Timpson, André G. Uitterlinden, Cornelia M. van Duijn, Frank J. A. van Rooij, Jaqueline M. Vink, Peter Vollenweider, Eero Vuoksimaa, Gérard Waeber, Nicholas J. Wareham, Nicole Warrington, Dawn Waterworth, Thomas Werge, H.-Erich Wichmann, Elisabeth Widen, Gonneke Willemsen, Alan F. Wright, Margaret J. Wright, Mousheng Xu, Jing Hua Zhao, Peter Kraft, David A. Hinds, Cecilia M. Lindgren, Reedik Mägi, Benjamin M. Neale, David M. Evans, and Sarah E. Medland </p>
<p class="paper"><i>Nature Human Behaviour</i> (2020)</p>
<p class="paper">
<strong>Summary: </strong>
A huge GWAS of handedness
</p>
<p class="paper">
<strong>Links: </strong>
<a href="https://www.nature.com/articles/s41562-020-00956-y">Nature Human Behaviour</a>
<a href="https://www.biorxiv.org/content/10.1101/831321v3">bioarxiv</a>
</p>
</div>
</div>
<hr>
</div>
<div>
<div class="grid">
<div class="g-col-2">
<h4 class="anchored">58</h4>
<img src="img/long_rates.png" class="thumbnail">
</div>
<div class="g-col-1"></div>
<div class="g-col-9">
<p></p>
<p class="paper">
<strong>Inferring Multidimensional Rates of Aging from Cross-Sectional Data</strong>
</p>
<p class="authors paper">E. Pierson, P.W. Koh, T. Hashimoto, D. Koller, J. Leskovec, N. Eriksson, and P. Liang</p>
<p class="paper"><i>Proc Mach Learn Res.</i> 2019 Apr; 89: 97–107.</p>
<p class="paper">
<strong>Summary: </strong>
Machine learning for inferring rates of aging
</p>
<p class="paper">
<strong>Links: </strong>
<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6752884/">PMC</a>
</p>
</div>
</div>
<hr>
</div>
<div>
<div class="grid">
<div class="g-col-2">
<h4 class="anchored">57</h4>
<img src="img/pd_self.png" class="thumbnail">
</div>
<div class="g-col-1"></div>
<div class="g-col-9">
<p></p>
<p class="paper">
<strong>Self-report data as a tool for subtype identification in genetically-defined Parkinson’s disease</strong>
</p>
<p class="authors paper">A.R. Winslow, C.L. Hyde, J.B. Wilk, N. Eriksson, P. Cannon, M.R. Miller, W.D. Hirst</p>
<p class="paper"><i>Scientific Reports</i> volume 8, Article number: 12992 (2018)</p>
<p class="paper">
<strong>Summary: </strong>
Validation of self-reported PD data
</p>
<p class="paper">
<strong>Links: </strong>
<a href="https://www.nature.com/articles/s41598-018-30843-6">Scientific Reports</a>
</p>
</div>
</div>
<hr>
</div>
<div>
<div class="grid">
<div class="g-col-2">
<h4 class="anchored">56</h4>
<img src="img/shared_brain.png" class="thumbnail">
</div>
<div class="g-col-1"></div>
<div class="g-col-9">
<p></p>
<p class="paper">
<strong>Analysis of shared heritability in common disorders of the brain</strong>
</p>
<p class="authors paper">Verneri Anttila, Brendan Bulik-Sullivan, Hilary K Finucane, Raymond K Walters, Jose Bras, Laramie Duncan, Valentina Escott-Price, Guido J Falcone, Padhraig Gormley, Rainer Malik, Nikolaos A Patsopoulos, Stephan Ripke, Zhi Wei, Dongmei Yu, Phil H Lee, Patrick Turley, Benjamin Grenier-Boley, Vincent Chouraki, Yoichiro Kamatani, Claudine Berr, Luc Letenneur, Didier Hannequin, Philippe Amouyel, Anne Boland, Jean-François Deleuze, Emmanuelle Duron, Badri N Vardarajan, Christiane Reitz, Alison M Goate, Matthew J Huentelman, M Ilyas Kamboh, Eric B Larson, Ekaterina Rogaeva, Peter St George-Hyslop, Hakon Hakonarson, Walter A Kukull, Lindsay A Farrer, Lisa L Barnes, Thomas G Beach, F Yesim Demirci, Elizabeth Head, Christine M Hulette, Gregory A Jicha, John SK Kauwe, Jeffrey A Kaye, James B Leverenz, Allan I Levey, Andrew P Lieberman, Vernon S Pankratz, Wayne W Poon, Joseph F Quinn, Andrew J Saykin, Lon S Schneider, Amanda G Smith, Joshua A Sonnen, Robert A Stern, Vivianna M Van Deerlin, Linda J Van Eldik, Denise Harold, Giancarlo Russo, David C Rubinsztein, Anthony Bayer, Magda Tsolaki, Petra Proitsi, Nick C Fox, Harald Hampel, Michael J Owen, Simon Mead, Peter Passmore, Kevin Morgan, Markus M Nöthen, Jonathan M Schott, Martin Rossor, Michelle K Lupton, Per Hoffmann, Johannes Kornhuber, Brian Lawlor, Andrew Mcquillin, Ammar Al-Chalabi, Joshua C Bis, Agustin Ruiz, Mercè Boada, Sudha Seshadri, Alexa Beiser, Kenneth Rice, Sven J van der Lee, Philip L De Jager, Daniel H Geschwind, Matthias Riemenschneider, Steffi Riedel-Heller, Jerome I Rotter, Gerhard Ransmayr, Bradley T Hyman, Carlos Cruchaga, Montserrat Alegret, Bendik Winsvold, Priit Palta, Kai-How Farh, Ester Cuenca-Leon, Nicholas Furlotte, Tobias Kurth, Lannie Ligthart, Gisela M Terwindt, Tobias Freilinger, Caroline Ran, Scott D Gordon, Guntram Borck, Hieab HH Adams, Terho Lehtimäki, Juho Wedenoja, Julie E Buring, Markus Schürks, Maria Hrafnsdottir, Jouke-Jan Hottenga, Brenda Penninx, Ville Artto, Mari Kaunisto, Salli Vepsäläinen, Nicholas G Martin, Grant W Montgomery, Mitja I Kurki, Eija Hämäläinen, Hailiang Huang, Jie Huang, Cynthia Sandor, Caleb Webber, Bertram Muller-Myhsok, Stefan Schreiber, Veikko Salomaa, Elizabeth Loehrer, Hartmut Göbel, Alfons Macaya, Patricia Pozo-Rosich, Thomas Hansen, Thomas Werge, Jaakko Kaprio, Andres Metspalu, Christian Kubisch, Michel D Ferrari, Andrea C Belin, Arn MJM van den Maagdenberg, John-Anker Zwart, Dorret Boomsma, N. Eriksson, Jes Olesen, Daniel I Chasman, Dale R Nyholt, Richard Anney, Andreja Avbersek, Larry Baum</p>
<p class="paper"><i>Science</i> 22 Jun 2018: Vol. 360, Issue 6395, eaap8757</p>
<p class="paper">
<strong>Summary: </strong>
Genetic sharing between 25 disorders of the brain
</p>
<p class="paper">
<strong>Links: </strong>
<a href="https://science.sciencemag.org/content/360/6395/eaap8757">Science</a>
</p>
</div>
</div>
<hr>
</div>
<div>
<div class="grid">
<div class="g-col-2">
<h4 class="anchored">55</h4>
<img src="img/light_pathways.png" class="thumbnail">
</div>
<div class="g-col-1"></div>
<div class="g-col-9">
<p></p>
<p class="paper">
<strong>Genome-wide association meta-analysis highlights light-induced signaling as a driver for refractive error</strong>
</p>
<p class="authors paper">Milly S Tedja, Robert Wojciechowski, Pirro G Hysi, N. Eriksson, Nicholas A Furlotte, Virginie JM Verhoeven, Adriana I Iglesias, Magda A Meester-Smoor, Stuart W Tompson, Qiao Fan, Anthony P Khawaja, Ching-Yu Cheng, René Höhn, Kenji Yamashiro, Adam Wenocur, Clare Grazal, Toomas Haller, Andres Metspalu, Juho Wedenoja, Jost B Jonas, Ya Xing Wang, Jing Xie, Paul Mitchell, Paul J Foster, Barbara EK Klein, Ronald Klein, Andrew D Paterson, S Mohsen Hosseini, Rupal L Shah, Cathy Williams, Yik Ying Teo, Yih Chung Tham, Preeti Gupta, Wanting Zhao, Yuan Shi, Woei-Yuh Saw, E-Shyong Tai, Xue Ling Sim, Jennifer E Huffman, Ozren Polašek, Caroline Hayward, Goran Bencic, Igor Rudan, James F Wilson, Peter K Joshi, Akitaka Tsujikawa, Fumihiko Matsuda, Kristina N Whisenhunt, Tanja Zeller, Peter J van der Spek, Roxanna Haak, Hanne Meijers-Heijboer, Elisabeth M van Leeuwen, Sudha K Iyengar, Jonathan H Lass, Albert Hofman, Fernando Rivadeneira, André G Uitterlinden, Johannes R Vingerling, Terho Lehtimäki, Olli T Raitakari, Ginevra Biino, Maria Pina Concas, Tae-Hwi Schwantes-An, Robert P Igo, Gabriel Cuellar-Partida, Nicholas G Martin, Jamie E Craig, Puya Gharahkhani, Katie M Williams, Abhishek Nag, Jugnoo S Rahi, Phillippa M Cumberland, Cécile Delcourt, Céline Bellenguez, Janina S Ried, Arthur A Bergen, Thomas Meitinger, Christian Gieger, Tien Yin Wong, Alex W Hewitt, David A Mackey, Claire L Simpson, Norbert Pfeiffer, Olavi Pärssinen, Paul N Baird, Veronique Vitart, Najaf Amin, Cornelia M van Duijn, Joan E Bailey-Wilson, Terri L Young, Seang-Mei Saw, Dwight Stambolian, Stuart MacGregor, Jeremy A Guggenheim, Joyce Y Tung, Christopher J Hammond, Caroline CW Klaver</p>
<p class="paper"><i>Nat Genet.</i> 2018; 50, 834-–848</p>
<p class="paper">
<strong>Summary: </strong>
161 loci for refractive error in a multi-ethnic cohort
</p>
<p class="paper">
<strong>Links: </strong>
<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5980758/">PMC</a>
<a href="https://www.nature.com/articles/s41588-018-0127-7">Nature Genetics</a>
</p>
</div>
</div>
<hr>
</div>
<div>
<div class="grid">
<div class="g-col-2">
<h4 class="anchored">54</h4>
<img src="img/mdd.png" class="thumbnail">
</div>
<div class="g-col-1"></div>
<div class="g-col-9">
<p></p>
<p class="paper">
<strong>Genome-wide association analyses identify 44 risk variants and refine the genetic architecture of major depression</strong>
</p>
<p class="authors paper">N.R. Wray, S. Ripke, M. Mattheisen, M. Trzaskowski, E.M. Byrne, A. Abdellaoui, M.J. Adams, E. Agerbo, T.M. Air, T.MF Andlauer, S-A. Bacanu, M. Bækvad-Hansen, A.FT Beekman, T.B. Bigdeli, E.B. Binder, D.RH Blackwood, J. Bryois, H.N Buttenschøn, J. Bybjerg-Grauholm, N. Cai, E. Castelao, J.H. Christensen, T-K Clarke, J.IR Coleman, L. Colodro-Conde, B. Couvy-Duchesne, N. Craddock, G.E. Crawford, C.A. Crowley, H.S. Dashti, G. Davies, I.J. Deary, F. Degenhardt, E.M. Derks, N. Direk, C.V. Dolan, E.C. Dunn, T.C. Eley, N. Eriksson, V. Escott-Price, F.H. Farhadi Kiadeh, H.K. Finucane, A.J Forstner, J. Frank, H.A. Gaspar, M. Gill, P. Giusti-Rodríguez, F.S. Goes, S.D. Gordon, J. Grove, L.S. Hall, E. Hannon, C.S. Hansen, T.F. Hansen, S. Herms, I.B. Hickie, P. Hoffmann, G. Homuth, C. Horn, J-J Hottenga, D.M. Hougaard, M. Hu, C.L. Hyde, M. Ising, R. Jansen, F. Jin, E. Jorgenson, J.A. Knowles, I.S. Kohane, J. Kraft, W.W. Kretzschmar, J. Krogh, Z. Kutalik, J.M. Lane, Y. Li, Y. Li, P.A. Lind, X. Liu, L. Lu, D.J. MacIntyre, D.F. MacKinnon, R.M. Maier, W. Maier, J. Marchini, H. Mbarek, P. McGrath, P. McGuffin, S.E. Medland, D. Mehta, C.M. Middeldorp, E. Mihailov, Y. Milaneschi, L. Milani, J. Mill, F.M. Mondimore, G.W. Montgomery, S. Mostafavi, N. Mullins, M. Nauck, B. Ng, M.G. Nivard, D.R. Nyholt, P.F. O’Reilly, H. Oskarsson, M.J. Owen, J.N. Painter, C.B. Pedersen, M.G. Pedersen, R.E. Peterson, E. Pettersson, W.J. Peyrot, G. Pistis, D. Posthuma, S.M. Purcell, J.A. Quiroz, P. Qvist, J.P. Rice, B.P. Riley, M. Rivera, S.S. Mirza, R. Saxena, R. Schoevers, E.C. Schulte, L. Shen, J. Shi, S.I. Shyn, E. Sigurdsson, G.BC Sinnamon, J.H. Smit, D.J. Smith, H. Stefansson, S. Steinberg, C.A. Stockmeier, F. Streit, J. Strohmaier, K.E. Tansey, H. Teismann, A. Teumer, W. Thompson, P.A. Thomson, T.E. Thorgeirsson, C. Tian, M. Traylor, J. Treutlein, V. Trubetskoy, A.G. Uitterlinden, D. Umbricht, S. Van der Auwera, A.M. van Hemert, A. Viktorin</p>
<p class="paper"><i>Nat Genet.</i> 2018; 50, 668--681</p>
<p class="paper">
<strong>Summary: </strong>
44 loci for major depression
</p>
<p class="paper">
<strong>Links: </strong>
<a href="http://dx.doi.org/10.1038/s41588-018-0090-3">Nature Genetics</a>
<a href="https://www.biorxiv.org/content/early/2017/07/24/167577">bioRxiv</a>
</p>
</div>
</div>
<hr>
</div>
<div>
<div class="grid">
<div class="g-col-2">
<h4 class="anchored">53</h4>
<img src="img/hla_infectious.png" class="thumbnail">
</div>
<div class="g-col-1"></div>
<div class="g-col-9">
<p></p>
<p class="paper">
<strong>Genome-wide association and HLA region fine-mapping studies identify susceptibility loci for multiple common infections</strong>
</p>
<p class="authors paper">C. Tian, B.S. Hromatka, A.K. Kiefer, N. Eriksson, S. Noble, J.Y. Tung, D.A. Hinds</p>
<p class="paper"><i>Nat Commun.</i> 2017; 8; 599</p>
<p class="paper">
<strong>Summary: </strong>
Many links between the HLA and infectious disease
</p>
<p class="paper">
<strong>Links: </strong>
<a href="http://dx.doi.org/10.1038/s41467-017-00257-5">Nature Communications</a>
<a href="http://www.biorxiv.org/content/early/2016/09/09/073056">bioRxiv</a>
</p>
</div>
</div>
<hr>
</div>
<div>
<div class="grid">
<div class="g-col-2">
<h4 class="anchored">52</h4>
<img src="img/adhd_sex.png" class="thumbnail">
</div>
<div class="g-col-1"></div>
<div class="g-col-9">
<p></p>
<p class="paper">
<strong>A genetic investigation of sex bias in the prevalence of attention deficit hyperactivity disorder</strong>
</p>
<p class="authors paper">J. Martin, R.K. Walters, D. Demontis, M. Mattheisen, S.H. Lee, E. Robinson, I. Brikell, L. Ghirardi, H. Larsson, P. Lichtenstein, N. Eriksson, 23andMe Research Team, Psychiatric Genomics Consortium: ADHD Subgroup, iPSYCH-Broad ADHD Workgroup, T. Werge, P.B. Mortensen, M.G. Pedersen, O. Mors, M. Nordentoft, D.M. Hougaard, J. Bybjerg-Grauholm, N. Wray, B. Franke, S.V. Faraone, M.C. O'Donovan, A. Thapar, A.D. Børglum, B.M. Neale</p>
<p class="paper"><i>bioRxiv</i> (2017)</p>
<p class="paper">
<strong>Summary: </strong>
Similar polygenic ADHD burden across sex.
</p>
<p class="paper">
<strong>Links: </strong>
<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5992329/">PubMed Central</a>
<a href="http://www.biorxiv.org/content/early/2017/06/23/154088">bioRxiv</a>
</p>
</div>
</div>
<hr>
</div>
<div>
<div class="grid">
<div class="g-col-2">
<h4 class="anchored">51</h4>
<img src="img/adhd.png" class="thumbnail">
</div>
<div class="g-col-1"></div>
<div class="g-col-9">
<p></p>
<p class="paper">
<strong>Discovery Of The First Genome-Wide Significant Risk Loci For ADHD</strong>
</p>
<p class="authors paper">D. Demontis, R.K. Walters, J. Martin, M. Mattheisen, T.D. Als, E. Agerbo, R. Belliveau, J. Bybjerg-Grauholm, M. Bækved-Hansen, F. Cerrato, K. Chambert, C. Churchhouse, A. Dumont, N. Eriksson, M. Gandal, J. Goldstein, J. Grove, C.S. Hansen, M. Hauberg, M. Hollegaard, D.P. Howrigan, H. Huang, J. Maller, A.R. Martin, J. Moran, J. Pallesen, D.S. Palmer, C.B. Pedersen, M.G. Pedersen, T. Poterba, J.B. Poulsen, S. Ripke, E.B. Robinson, F.K. Satterstrom, C. Stevens, P. Turley, H. Won, ADHD Working Group of the Psychiatric Genomics Con, Early Lifecourse and Genetic Epidemiology (EAGLE), 23andMe Research Team, O.A. Andreassen, C. Burton, D. Boomsma, B. Cormand, S. Dalsgaard, B. Franke, J. Gelernter, D. Geschwind, H. Hakonarson, J. Haavik, H. Kranzler, J. Kuntsi, K. Langley, K-P. Lesch, C. Middeldorp, A. Reif, L.A. Rohde, P. Roussos, R. Schachar, P. Sklar, E. Sonuga-Barke, P.F. Sullivan, A. Thapar, J.Y. Tung, I. Waldman, M. Nordentoft, D.M. Hougaard, T. Werge, O. Mors, P.B. Mortensen, M.J. Daly, S.V. Faraone, A.D. Børglum, B.M. Neale</p>
<p class="paper"><i>Nature Genetics</i> (2019)</p>
<p class="paper">
<strong>Summary: </strong>
12 loci for ADHD.
</p>
<p class="paper">
<strong>Links: </strong>
<a href="https://www.nature.com/articles/s41588-018-0269-7">Nature Genetics</a>
<a href="http://www.biorxiv.org/content/early/2017/06/03/145581">bioRxiv</a>
</p>
</div>
</div>
<hr>
</div>
<div>
<div class="grid">
<div class="g-col-2">
<h4 class="anchored">50</h4>
<img src="img/psoriasis2.png" class="thumbnail">
</div>
<div class="g-col-1"></div>
<div class="g-col-9">
<p></p>
<p class="paper">
<strong>Large scale meta-analysis characterizes genetic architecture for common psoriasis associated variants</strong>
</p>
<p class="authors paper">L.C. Tsoi, P.E. Stuart, C. Tian, J.E. Gudjonsson, S. Das, M. Zawistowski, E. Ellinghaus, J.N. Barker, V. Chandran, N. Dand, K.C. Duffin, C. Enerbäck, T. Esko, A. Franke, D.D. Gladman, P. Hoffmann, K. Kingo, S. Kõks, G. G. Krueger, H.W. Lim, A. Metspalu, U. Mrowietz, S. Mucha, P. Rahman, A. Reis, T. Tejasvi, R. Trembath, J.J. Voorhees, S. Weidinger, M. Weichenthal, X. Wen, N. Eriksson, H.M. Kang, D.A. Hinds, R.P. Nair, G.R. Abecasis, J.T. Eldera</p>
<p class="paper"><i>Nat Commun.</i> 2017; 8: 15382</p>
<p class="paper">
<strong>Summary: </strong>
16 new associations with psoriasis
</p>
<p class="paper">
<strong>Links: </strong>
<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5458077/">PMC</a>
</p>
</div>
</div>
<hr>
</div>
<div>
<div class="grid">
<div class="g-col-2">
<h4 class="anchored">49</h4>
<img src="img/afb.png" class="thumbnail">
</div>
<div class="g-col-1"></div>
<div class="g-col-9">
<p></p>
<p class="paper">
<strong>Genome-wide analysis identifies 12 loci influencing human reproductive behavior</strong>
</p>
<p class="authors paper">N. Barban, R. Jansen, R. de Vlaming, A. Vaez, J.J. Mandemakers, F.C. Tropf, X. Shen, J.F. Wilson, D.I. Chasman, I.M. Nolte, V. Tragante, S.W. van der Laan, JRB Perry, A. Kong, T. S Ahluwalia, E. Albrecht, L. Yerges-Armstrong, G. Atzmon, K. Auro, K. Ayers, A. Bakshi, D. Ben-Avraham, K. Berger, A. Bergman, L. Bertram, L.F. Bielak, G. Bjornsdottir, M. Jan Bonder, L. Broer, M. Bui, C. Barbieri, A. Cavadino, J.E. Chavarro, C. Turman, M. Pina Concas, H.J. Cordell, G. Davies, P. Eibich, N. Eriksson, T. Esko, J. Eriksson, F. Falahi, J.F. Felix, M.A. Fontana, L. Franke, I. Gandin, A.J. Gaskins, C. Gieger, E. P Gunderson, X. Guo, C. Hayward, C. He, E. Hofer, H. Huang, P.K. Joshi, S. Kanoni, R. Karlsson, S. Kiechl, A. Kifley, A. Kluttig, P. Kraft, V. Lagou, C. Lecoeur, J. Lahti, R. Li-Gao, P. A Lind, T. Liu, E. Makalic, C. Mamasoula, L. Matteson, H. Mbarek, P.F. McArdle, G. McMahon, SFW Meddens, E. Mihailov, M. Miller, S.A. Missmer, C. Monnereau, P.J. van der Most, R. Myhre, M.A Nalls, T. Nutile, I. Panagiota Kalafati, E. Porcu, I. Prokopenko, K.B. Rajan, J. Rich-Edwards, C.A. Rietveld, A. Robino, L.M. Rose, R. Rueedi, K.A. Ryan, Y. Saba, D. Schmidt, J.A. Smith, L. Stolk, E. Streeten, A. Tönjes, G. Thorleifsson, S. Ulivi, J. Wedenoja, J. Wellmann, P. Willeit, J. Yao, L. Yengo, J. Hua Zhao, W. Zhao, D.V. Zhernakova, N. Amin, H. Andrews, B. Balkau, N. Barzilai, S. Bergmann, G. Biino, H. Bisgaard, K. Bønnelykke, D. I Boomsma, J.E. Buring, H. Campbell, S. Cappellani, M. Ciullo, S.R. Cox, F. Cucca, D. Toniolo, G. Davey-Smith, I.J. Deary, G. Dedoussis, P. Deloukas, C.M. van Duijn, EJC de Geus, J.G. Eriksson, D.A. Evans, J.D. Faul, C. Felicita Sala, P. Froguel, P. Gasparini, G. Girotto, H-J Grabe, K. Halina Greiser, PJF Groenen, H.G. de Haan, J. Haerting, T.B. Harris, A.C. Heath, K. Heikkilä, A. Hofman, G. Homuth, E.G. Holliday, J. Hopper, E. Hyppönen</p>
<p class="paper"><i>Nature Genetics</i> 48, 1462–1472 (2016)</p>
<p class="paper">
<strong>Summary: </strong>
GWAS of age at first birth / number of children.
</p>
<p class="paper">
<strong>Links: </strong>
<a href="http://www.nature.com/ng/journal/v48/n12/full/ng.3698.html">Nature Genetics</a>
</p>
</div>
</div>
<hr>
</div>
<div>
<div class="grid">
<div class="g-col-2">
<h4 class="anchored">48</h4>
<img src="img/jak2.png" class="thumbnail">
</div>
<div class="g-col-1"></div>
<div class="g-col-9">
<p></p>
<p class="paper">
<strong>Germline variants predispose to both JAK2 V617F clonal hematopoiesis and myeloproliferative neoplasms</strong>
</p>
<p class="authors paper">D.A. Hinds, K.E. Barnholt, R.A. Mesa, A.K. Kiefer, C.B. Do, N. Eriksson, J.L. Mountain, U. Francke, J.Y. Tung, H.M. Nguyen, H. Zhang, L. Gojenola, J.L. Zehnder, J. Gotlib</p>
<p class="paper"><i>Blood</i> (2016) doi:10.1182/blood-2015-06-652941</p>
<p class="paper">
<strong>Summary: </strong>
Germline variants predisposing to myeloproliferative neoplasms and to JAK2 V617F clonal hematopoiesis.
</p>
<p class="paper">
<strong>Links: </strong>
<a href="http://www.bloodjournal.org/content/early/2016/06/30/blood-2015-06-652941">Blood</a>
</p>
</div>
</div>
<hr>
</div>
<div>
<div class="grid">
<div class="g-col-2">
<h4 class="anchored">47</h4>
<img src="img/migraine.png" class="thumbnail">
</div>
<div class="g-col-1"></div>
<div class="g-col-9">
<p></p>
<p class="paper">
<strong>Meta-analysis of 375,000 individuals identifies 38 susceptibility loci for migraine</strong>
</p>
<p class="authors paper">P. Gormley, V. Anttila, B.S. Winsvold, P. Palta, T. Esko, T.H. Pers, K-H. Farh, E. Cuenca-Leon, M. Muona, N.A. Furlotte, T. Kurth, A. Ingason, G. McMahon, L. Ligthart, G.M. Terwindt, M. Kallela, T.M. Freilinger, C. Ran, S.G. Gordon, A.H. Stam, S. Steinberg, G. Borck, M. Koiranen, L. Quaye, H.H.H. Adams, T. Lehtimäki, A-P. Sarin, J. Wedenoja, D.A. Hinds, J.E. Buring, M. Schürks, P.M. Ridker, M.G. Hrafnsdottir, H. Stefansson, S.M. Ring, J-J. Hottenga, B.W.J.H. Penninx, M. Färkkilä, V. Artto, M. Kaunisto, S. Vepsäläinen, R. Malik, A.C. Heath, P.A.F. Madden, N.G. Martin, G.W. Montgomery, M.I. Kurki, M. Kals, R. Mägi, K. Pärn, E. Hämäläinen, H. Huang, A.E. Byrnes, L. Franke, J. Huang, E. Stergiakouli, P.H. Lee, C. Sandor, C. Webber, Z. Cader, B. Muller-Myhsok, S. Schreiber, T. Meitinger, J.G. Eriksson, V. Salomaa, K. Heikkilä, E. Loehrer, A.G. Uitterlinden, A. Hofman, C.M. van Duijn, L. Cherkas, L.M. Pedersen, A. Stubhaug, C.S. Nielsen, M. Männikkö, E. Mihailov, L. Milani, H. Göbel, A-L. Esserlind, A.F. Christensen, T.F. Hansen, T. Werge, V. Anttila, V. Artto, A.C. Belin, D.I. Boomsma, S. Børte, L. Cherkas, A.F. Christensen, B. Cormand, E. Cuenca-Leon, G.D. Smith, M. Dichgans, C. van Duijn, E. Eising, T. Esko, A-L. Esserlind, M. Ferrari, R.R Frants, T. M Freilinger, L. Griffiths, E. Hamalainen, T.F. Hansen, M. Hiekkala, M.A. Ikram, A. Ingason, M-R. Järvelin, R. Kajanne, M. Kallela, J. Kaprio, M. Kaunisto, C. Kubisch, M. Kurki, T. Kurth, L. Launer, T. Lehtimaki, D. Lessel, L. Ligthart, N. Litterman, A.M.J.M.van den Maagdenberg, A. Macaya, R. Malik, M. Mangino, G. McMahon, B. Muller-Myhsok, C. Northover, J. Olesen, L.M. Pedersen, N. Pedersen, D. Posthuma, P. Pozo-Rosich, A. Pressman, L. Quaye, O. Raitakari, M. Schürks, C. Sintas, H. Stefansson, S. Steinberg, D. Strachan, G.M. Terwindt, M. Vila-Pueyo, M. Wessman, B.S. Winsvold, W. Wrenthal, H. Zhao, J-A. Zwart, J. Kaprio, A.J. Aromaa, O. Raitakari, M.A. Ikram, T. Spector, M-R. Järvelin, A. Metspalu, C. Kubisch, D.P. Strachan, M.D. Ferrari, A.C. Belin, M. Dichgans, M. Wessman, A.M.J.M. van den Maagdenberg, J-A. Zwart, D.I. Boomsma, G.D. Smith, K. Stefansson, N. Eriksson, M.J. Daly, B.M. Neale, J. Olesen, D.I. Chasman, D.R. Nyholt, A. Palotie</p>
<p class="paper"><i>Nature Genetics</i> (2016) doi:10.1038/ng.3598</p>
<p class="paper">
<strong>Summary: </strong>
A huge GWAS of migraines.
</p>
<p class="paper">
<strong>Links: </strong>
<a href="https://www.nature.com/articles/ng.3598">Nature Genetics</a>
<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5331903/">PubMed Central</a>
</p>
</div>
</div>
<hr>
</div>
<div>
<div class="grid">
<div class="g-col-2">
<h4 class="anchored">46</h4>
<img src="img/morning.png" class="thumbnail">
</div>
<div class="g-col-1"></div>
<div class="g-col-9">
<p></p>
<p class="paper">
<strong>GWAS of 89,283 individuals identifies genetic variants associated with self-reporting of being a morning person</strong>
</p>
<p class="authors paper">Y. Hu, A. Shmygelska, D. Tran, N. Eriksson, J.Y. Tung, D.A. Hinds</p>
<p class="paper"><i>Nature Communications</i> 7, Article number: 10448, Feb 2016.</p>
<p class="paper">
<strong>Summary: </strong>
On the genetics of circadian rhythms
</p>
<p class="paper">
<strong>Links: </strong>
<a href="http://nature.com/articles/doi:10.1038/ncomms10448">Nature Communications</a>
</p>
<p class="paper">
<strong>Press: </strong>
<a href="http://www.theverge.com/2016/2/2/10893994/23andme-morning-person-dna-genes">The Verge</a>
</p>
</div>
</div>
<hr>
</div>
<div>
<div class="grid">
<div class="g-col-2">
<h4 class="anchored">45</h4>
<img src="img/mooc.png" class="thumbnail">
</div>
<div class="g-col-1"></div>
<div class="g-col-9">
<p></p>
<p class="paper">
<strong>Who's Benefiting from MOOCs, and Why</strong>
</p>
<p class="authors paper">C. Zhenghao, B. Alcorn, G. Christensen, N. Eriksson, D. Koller, E.J. Emanuel</p>
<p class="paper"><i>Harvard Business Review</i>, September 22, 2015.</p>
<p class="paper">
<strong>Summary: </strong>
Among learners who complete courses, MOOCs do have a real impact.
</p>
<p class="paper">
<strong>Links: </strong>
<a href="https://hbr.org/2015/09/whos-benefiting-from-moocs-and-why">Harvard Business Review</a>
</p>
<p class="paper">
<strong>Press: </strong>
<a href="http://www.ft.com/cms/s/2/fe42ba48-5d31-11e5-9846-de406ccb37f2.html#axzz3mTzTuer2">Financial Times</a>
<a href="http://www.usatoday.com/story/tech/2015/09/22/coursera-survey-finds-its-courses-boon-emerging-economy-workers/72563786/">USA Today</a>
<a href="http://www.inc.com/graham-winfrey/how-online-courses-can-help-entrepreneurs-close-the-skills-gap.html">Inc.</a>
<a href="http://chronicle.com/blogs/wiredcampus/what-the-results-of-a-survey-of-coursera-students-mean-for-online-learning/57401">Chronicle of Higher Education</a>
</p>
</div>
</div>
<hr>
</div>
<div>
<div class="grid">
<div class="g-col-2">
<h4 class="anchored">44</h4>
<img src="img/virtual_pd.png" class="thumbnail">
</div>
<div class="g-col-1"></div>
<div class="g-col-9">
<p></p>
<p class="paper">
<strong>Virtual research visits and direct-to-consumer genetic testing in Parkinson’s disease</strong>
</p>
<p class="authors paper">E.R. Dorsey, K.C. Darwin, S. Mohammed, S. Donohue, A. Tethal, M.A. Achey, S. Ward, E. Caughey, E.D. Conley, N. Eriksson, B. Ravina</p>
<p class="paper"><i>Digital Health</i>, Jun 2015.</p>
<p class="paper">
<strong>Summary: </strong>
In depth, virtual visits of Parkinson's disease patients.
</p>
<p class="paper">
<strong>Links: </strong>
<a href="https://journals.sagepub.com/doi/full/10.1177/2055207615592998">Digital Health</a>
<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5999055/">PubMed Central</a>
</p>
</div>
</div>
<hr>
</div>
<div>
<div class="grid">
<div class="g-col-2">
<h4 class="anchored">43</h4>
<img src="img/rosacea.png" class="thumbnail">
</div>
<div class="g-col-1"></div>
<div class="g-col-9">
<p></p>
<p class="paper">
<strong>Assessment of the Genetic Basis of Rosacea by Genome-Wide Association Study</strong>
</p>
<p class="authors paper">A. L. S. Chang, I. Raber, J. Xu, R. Li, R. Spitale, J. Chen, A. K. Kiefer, C. Tian, N. Eriksson, D. A. Hinds, J. Y. Tung</p>
<p class="paper"><i>Journal of Investigative Dermatology</i>, March 2015.</p>
<p class="paper">
<strong>Summary: </strong>
HLA associations with rosacea
</p>
<p class="paper">
<strong>Links: </strong>
<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4434179/">PubMed Central</a>
<a href="https://www.jidonline.org/article/S0022-202X(15)37273-0/fulltext">JID</a>
</p>
</div>
</div>
<hr>
</div>
<div>
<div class="grid">
<div class="g-col-2">
<h4 class="anchored">42</h4>
<img src="img/crossover.png" class="thumbnail">
</div>
<div class="g-col-1"></div>
<div class="g-col-9">
<p></p>
<p class="paper">
<strong>Escape from crossover interference increases with maternal age</strong>
</p>
<p class="authors paper">C. L. Campbell, N. A. Furlotte, N. Eriksson, D. A. Hinds, A. Auton</p>
<p class="paper"><i>Nature Communications</i>, 6, Feb 2015.</p>
<p class="paper">
<strong>Summary: </strong>
Analysis of recombination rate and hotspot usage with respect to age.
</p>
<p class="paper">
<strong>Links: </strong>
<a href="http://www.nature.com/ncomms/2015/150219/ncomms7260/full/ncomms7260.html">Nature Communications</a>
<a href="http://arxiv.org/abs/1408.5517">arxiv</a>
</p>
</div>
</div>
<hr>
</div>
<div>
<div class="grid">
<div class="g-col-2">
<h4 class="anchored">41</h4>
<img src="img/motionsick.png" class="thumbnail">
</div>
<div class="g-col-1"></div>
<div class="g-col-9">
<p></p>
<p class="paper">
<strong>Genetic variants associated with motion sickness point to roles for inner ear development, neurological processes, and glucose homeostasis</strong>
</p>
<p class="authors paper">B. S. Hromatka, J. Y. Tung, A. K. Kiefer, C. B. Do, D. A. Hinds, N. Eriksson</p>
<p class="paper"><i>Human Molecular Genetics</i>, 2015.</p>
<p class="paper">
<strong>Summary: </strong>
The first 35 variants associated with motion sickness.
</p>
<p class="paper">
<strong>Links: </strong>
<a href="http://hmg.oxfordjournals.org/content/early/2015/02/17/hmg.ddv028.full">HMG</a>
<a href="http://biorxiv.org/content/early/2014/02/04/002386">biorxiv</a>
<a href="http://www.vox.com/2015/8/9/9121583/motion-sickness-carsick">Vox</a>
</p>
</div>
</div>
<hr>
</div>
<div>
<div class="grid">
<div class="g-col-2">
<h4 class="anchored">40</h4>
<img src="img/educational.png" class="thumbnail">
</div>
<div class="g-col-1"></div>
<div class="g-col-9">
<p></p>
<p class="paper">
<strong>Replicability and Robustness of Genome-Wide-Association Studies for Behavioral Traits</strong>
</p>
<p class="authors paper">C. A. Rietveld, D. Conley, N. Eriksson, T. Esko, S. E. Medland, A. A. E. Vinkhuyzen, J. Yang, J. D. Boardman, C. F. Chabris, C. T. Dawes, B. W. Domingue, D. A. Hinds, M. Johannesson, A. K. Kiefer, D. Laibson, P. K. E. Magnusson, J. L. Mountain, S. Oskarsson, O. Rostapshova, A. Teumer, J. Y. Tung, P. M. Visscher, D. J. Benjamin, D. Cesarini, P. D. Koellinger</p>
<p class="paper"><i>Psychological Science,</i> Nov 2014, vol. 25, no. 11, 1975-1986.</p>
<p class="paper">
<strong>Summary: </strong>
Replication of educational attainment associations
</p>
<p class="paper">
<strong>Links: </strong>
<a href="http://pss.sagepub.com/content/25/11/1975.short">Psych Sci</a>
</p>
</div>
</div>
<hr>
</div>
<div>
<div class="grid">
<div class="g-col-2">
<h4 class="anchored">39</h4>
<img src="img/neuroX.png" class="thumbnail">
</div>
<div class="g-col-1"></div>
<div class="g-col-9">
<p></p>
<p class="paper">
<strong>NeuroX, a Fast and Efficient Genotyping Platform for Investigation of Neurodegenerative Diseases</strong>
</p>
<p class="authors paper">M. A. Nalls, J. Bras, D. G. Hernandez, M. F. Keller, E. Majounie, A. E. Renton, M. Saad, I. Jansen, R. Guerreiro, S. Lubbe, V. Plagnol, J. R. Gibbs, C. Schulte, N. Pankratz, M. Sutherland, L. Bertram, C. M. Lill, A. L. DeStefano, T. Faroud, N. Eriksson, J. Y. Tung, C. Edsall, N. Nichols, J. Brooks, S. Arepalli, H. Pliner, C. Letson, P. Heutink, M. Martinez, T. Gasser, B. J. Traynor, N. Wood, J. Hardy, A. B. Singleton</p>
<p class="paper"><i>Neurobiology of Aging,</i> 4 Aug 2014.</p>
<p class="paper">
<strong>Summary: </strong>
A cheap genotyping chip for neurodegenerative disease research.
</p>
<p class="paper">
<strong>Links: </strong>
<a href="http://dx.doi.org/10.1016/j.neurobiolaging.2014.07.028">DOI</a>
</p>
</div>
</div>
<hr>
</div>
<div>
<div class="grid">
<div class="g-col-2">
<h4 class="anchored">38</h4>
<img src="img/mega-meta.png" class="thumbnail">
</div>
<div class="g-col-1"></div>
<div class="g-col-9">
<p></p>
<p class="paper">
<strong>Large-scale meta-analysis of genome-wide association data identifies six new risk loci for Parkinson's disease</strong>
</p>
<p class="authors paper">M. A. Nalls,   N. Pankratz,    C. M. Lill,   C. B. Do,    D. G. Hernandez,   M. Saad,   A. L. DeStefano,  E. Kara,   J. Bras,  M. Sharma,    C. Schulte,    M. F. Keller,   S. Arepalli,   C. Letson, C. Edsall,  H. Stefansson,  X. Liu, H. Pliner,  J. H. Lee,   R. Cheng, International Parkinson's Disease Genomics Consortium (IPDGC),  Parkinson's Study Group (PSG) Parkinson's Research: The Organized GENetics Initiative (PROGENI),    23andMe,    GenePD, NeuroGenetics Research Consortium (NGRC),   Hussman Institute of Human Genomics (HIHG), The Ashkenazi Jewish Dataset Investigator,  Cohorts for Health and Aging Research in Genetic Epidemiology (CHARGE), North American Brain Expression Consortium (NABEC), United Kingdom Brain Expression Consortium (UKBEC), Greek Parkinson's Disease Consortium,   Alzheimer Genetic Analysis Group,   M. A. Ikram,  J. P. A. Ioannidis, G. M Hadjigeorgiou, J. C. Bis, M. Martinez, J. S. Perlmutter,  A. Goate,   K. Marder,   B. Fiske, M. Sutherland,   G. Xiromerisiou,   R. H. Myers,    L. N. Clark,   K. Stefansson,    J. A. Hardy,   P. Heutink,  H. Chen,   N. W. Wood,    H. Houlden, H. Payami,   A. Brice,   W. K. Scott,    T. Gasser,  L. Bertram,   N. Eriksson,  T. Foroud, A. B. Singleton</p>
<p class="paper"><i>Nature Genetics</i> (2014) doi:10.1038/ng.3043</p>
<p class="paper">
<strong>Summary: </strong>
Six new loci for Parkinson's disease in the largest meta-analysis to date
</p>
<p class="paper">
<strong>Links: </strong>
<a href="https://www.nature.com/articles/ng.3043">Nature Genetics</a>
<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4146673/">PubMed Central</a>
</p>
</div>
</div>
<hr>
</div>
<div>
<div class="grid">
<div class="g-col-2">
<h4 class="anchored">37</h4>
<img src="img/haploscore.png" class="thumbnail">
</div>
<div class="g-col-1"></div>
<div class="g-col-9">
<p></p>
<p class="paper">
<strong>Reducing pervasive false positive identical-by-descent segments detected by large-scale pedigree analysis</strong>
</p>
<p class="authors paper">E. Y. Durand, N. Eriksson, C. Y. McLean</p>
<p class="paper"><i>Mol Biol Evol</i>, 30 April 2014.</p>
<p class="paper">
<strong>Summary: </strong>
Improving the accuracy of germline on short IBD segments
</p>
<p class="paper">
<strong>Links: </strong>
<a href="http://mbe.oxfordjournals.org/content/early/2014/04/30/molbev.msu151.abstract">MBE</a>
<a href="http://arxiv.org/abs/1311.1120">arXiv</a>
</p>
</div>
</div>
<hr>
</div>
<div>
<div class="grid">
<div class="g-col-2">
<h4 class="anchored">36</h4>
<img src="img/hayfever.png" class="thumbnail">
</div>
<div class="g-col-1"></div>
<div class="g-col-9">
<p></p>
<p class="paper">
<strong>Genome-wide association analysis identifies 11 risk variants associated with the asthma with hay fever phenotype</strong>
</p>
<p class="authors paper">M. A. R. Ferreira, M. C. Matheson, C. S. Tang, R. Granell, W. Ang, J. Hui, A. K. Kiefer, D. L. Duffy, S. Baltic, P. Danoy, M. Bui, L. Price, P. D. Sly, N. Eriksson, P. A. Madden, M. J. Abramson, P. G. Holt, A. C. Heath, M. Hunter, B. Musk, C. F. Robertson, P. Le Souef, W. Montgomery, A.J. Henderson, J. Y. Tung, S. C. Dharmage, M. A. Brown, A. James, P. J. Thompson, C. Pennell, N. G. Martin, D. M.  Evans, D. A. Hinds, J. L. Hopper</p>
<p class="paper"><i>Journal of Allergy and Clinical Immunology</i>, 31 December 2013</p>
<p class="paper">
<strong>Summary: </strong>
Meta-analysis of hay fever with asthma
</p>
<p class="paper">
<strong>Links: </strong>
<a href="http://dx.doi.org/10.1016/j.jaci.2013.10.030">JACI</a>
<a href="http://www.sciencedirect.com/science/article/pii/S0091674913016436">ScienceDirect</a>
</p>
</div>
</div>
<hr>
</div>
<div>
<div class="grid">
<div class="g-col-2">
<h4 class="anchored">35</h4>
<img src="img/gradiant.png" class="thumbnail">
</div>
<div class="g-col-1"></div>
<div class="g-col-9">
<p></p>
<p class="paper">
<strong>Gradiant Boosting as a SNP filter: an evaluation using simulated and hair morphology data</strong>
</p>
<p class="authors paper">G. H. Lubke, C. Laurin, R. Walters, N. Eriksson, P. Hysi, T. D. Spector, G. W. Montgomery, D. I. Boomsma, N. G. Martin, and S. E. Medland</p>
<p class="paper"><i>Journal of Data Mining in Genomics &amp; Proteomics</i>, 2013, 4:4</p>
<p class="paper">
<strong>Summary: </strong>
Gradiant boosting applied to GWAS data
</p>
<p class="paper">
<strong>Links: </strong>
<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3882018/">PMC</a>
</p>
</div>
</div>
<hr>
</div>
<div>
<div class="grid">
<div class="g-col-2">
<h4 class="anchored">34</h4>
<img src="img/allergy.png" class="thumbnail">
</div>
<div class="g-col-1"></div>
<div class="g-col-9">
<p></p>
<p class="paper">
<strong>A genome-wide association meta-analysis of self-reported allergy identifies shared and allergy-specific susceptibility loci</strong>
</p>
<p class="authors paper">D. A. Hinds, G. McMahon, A. K. Kiefer, C. B. Do, N. Eriksson, D. M. Evans, B. St Pourcain, S. M. Ring, J. L. Mountain, U. Francke, G. Davey-Smith, N. J. Timpson, and J. Y. Tung</p>
<p class="paper"><i>Nat Genet</i>, 2013 June 30.</p>
<p class="paper">
<strong>Summary: </strong>
Sixteen associations with allergies, eight of them shared with asthma
</p>
<p class="paper">
<strong>Links: </strong>
<a href="http://dx.doi.org/10.1038/ng.2686">Nature Genetics</a>
</p>
</div>
</div>
<hr>
</div>
<div>
<div class="grid">
<div class="g-col-2">
<h4 class="anchored">33</h4>
<img src="img/pdiron.png" class="thumbnail">
</div>
<div class="g-col-1"></div>
<div class="g-col-9">
<p></p>
<p class="paper">
<strong>Serum iron levels and the risk of Parkinson's disease: a Mendelian randomization study</strong>
</p>
<p class="authors paper">I. Pichler, F. Del Greco M., M. Gogele, C. M. Lill, L. Bertram, C. B. Do, N. Eriksson, T. Foroud, R. H. Myers, M. Nalls, M. F. Keller, B. Benyamin, J. B. Whitfield, P. P. Pramstaller, A. A. Hicks, J. Thompson, and C. Minelli</p>
<p class="paper"><i>PLOS Med.</i> 10(6): e1001462.</p>
<p class="paper">
<strong>Summary: </strong>
Mendelian randomization study shows that higher iron levels are protective for Parkinson's disease
</p>
<p class="paper">
<strong>Links: </strong>
<a href="https://journals.plos.org/plosmedicine/article?id=10.1371/journal.pmed.1001462">PLOS</a>
<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3672214/">PMC</a>
</p>
</div>
</div>
<hr>
</div>
<div>
<div class="grid">
<div class="g-col-2">
<h4 class="anchored">32</h4>
<img src="img/stretch.png" class="thumbnail">
</div>
<div class="g-col-1"></div>
<div class="g-col-9">
<p></p>
<p class="paper">
<strong>Genome-Wide Association Analysis Implicates Elastic Microfibrils in the Development of Nonsyndromic Striae Distensae</strong>
</p>
<p class="authors paper">J. Y. Tung, A.K. Kiefer, M. Mullins, U. Francke, and N. Eriksson</p>
<p class="paper"><i>J Invest Dermatol</i>, 2013 Apr 30</p>
<p class="paper">
<strong>Summary: </strong>
Variants near the elastin gene influence stretch marks
</p>
<p class="paper">
<strong>Links: </strong>
<a href="http://dx.doi.org/10.1038/jid.2013.196">JID</a>
</p>
</div>
</div>
<hr>
</div>
<div>
<div class="grid">
<div class="g-col-2">
<h4 class="anchored">31</h4>
<img src="img/myopia.png" class="thumbnail">
</div>
<div class="g-col-1"></div>
<div class="g-col-9">
<p></p>
<p class="paper">
<strong>Genome-wide analysis points to roles for extracellular matrix remodeling, the visual cycle, and neuronal development in myopia</strong>
</p>
<p class="authors paper">A. K. Kiefer, J. Y. Tung, C. B. Do, D. A. Hinds, J. L. Mountain, U. Francke, and N. Eriksson</p>
<p class="paper"><i>PLoS Genet.</i>, 9(2): e1003299.</p>
<p class="paper">
<strong>Summary: </strong>
Twenty-two associations for myopia age of onset, many of which are near genes with interesting roles in vision development.
</p>
<p class="paper">
<strong>Links: </strong>
<a href="https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1003299">PLoS</a>
<a href="http://arxiv.org/abs/1209.2089">arXiv</a>
</p>
</div>
</div>
<hr>
</div>
<div>
<div class="grid">
<div class="g-col-2">
<h4 class="anchored">30</h4>
<img src="img/brca.png" class="thumbnail">
</div>
<div class="g-col-1"></div>
<div class="g-col-9">
<p></p>
<p class="paper">
<strong>Dealing with the unexpected: Consumer responses to direct-access *BRCA* mutation testing</strong>
</p>
<p class="authors paper">U. Francke, C. Dijamco, A. K. Kiefer, N. Eriksson, B. R. Moiseff, J. Y. Tung, and J. L. Mountain</p>
<p class="paper"><i>PeerJ</i>, 1:e8.</p>
<p class="paper">
<strong>Summary: </strong>
How do people respond to learning their BRCA status via 23andMe?
</p>
<p class="paper">
<strong>Links: </strong>
<a href="https://peerj.com/articles/8">PeerJ</a>
</p>
</div>
</div>
<hr>
</div>
<div>
<div class="grid">
<div class="g-col-2">
<h4 class="anchored">29</h4>
<img src="img/aa.png" class="thumbnail">
</div>
<div class="g-col-1"></div>
<div class="g-col-9">
<p></p>
<p class="paper">
<strong>Androgenetic alopecia: identification of four new genetic risk loci and evidence for the contribution of WNT-signaling to its etiology</strong>
</p>
<p class="authors paper">S. Heilmann, A. K. Kiefer, N. Fricker, D. Drichel, A. M. Hillmer, C. Herold, J. Y. Tung, N. Eriksson, S. Redler, R. C. Betz, R. Li, A. Karason, D. R. Nyholt, K. Song, S. H. Vermeulen, S. Kanoni, G. Dedoussis, N. G. Martin, L. A. Kiemeney, V. Mooser, K. Stefansson, J. B. Richards, T. Becker, F. F. Brockschmidt, D. A. Hinds, and M. M. Nothen</p>
<p class="paper"><i>J Invest Dermatol</i>, 2013 Jan 28.</p>
<p class="paper">
<strong>Summary: </strong>
Four new loci for male-pattern baldness.
</p>
<p class="paper">
<strong>Links: </strong>
<a href="http://dx.doi.org/10.1038/jid.2013.43">JID</a>
</p>
</div>
</div>
<hr>
</div>
<div>
<div class="grid">
<div class="g-col-2">
<h4 class="anchored">28</h4>
<img src="img/cilantro.png" class="thumbnail">
</div>
<div class="g-col-1"></div>
<div class="g-col-9">
<p></p>
<p class="paper">
<strong>A genetic variant near olfactory receptor genes influences cilantro preference</strong>
</p>
<p class="authors paper">N. Eriksson, S. Wu, C. B. Do, A. K. Kiefer, J. Y. Tung, J. L.  Mountain, D. A. Hinds, and U. Francke</p>
<p class="paper"><i>Flavour</i>, 1:22, Dec 2012.</p>
<p class="paper">
<strong>Summary: </strong>
We uncover a cluster of olfactory receptor genes that play a role in cilantro dislike.
</p>
<p class="paper">
<strong>Links: </strong>
<a href="http://www.flavourjournal.com/content/1/1/22/abstract">Flavour</a>
<a href="http://arxiv.org/abs/1209.2096">arXiv</a>
</p>
<p class="paper">
<strong>Press: </strong>
<a href="http://www.nature.com/news/soapy-taste-of-coriander-linked-to-genetic-variants-1.11398">Nature</a>
<a href="http://www.npr.org/blogs/thesalt/2012/09/14/161057954/love-to-hate-cilantro-its-in-your-genes-and-maybe-in-your-head">NPR</a>
<a href="http://gizmodo.com/5942551/genetic-proof-that-you-really-do-hate-cilantro">Gizmodo</a>
<a href="http://boingboing.net/2012/09/12/why-cilantro-haters-hate-cilan.html">BoingBoing</a>
<a href="http://www.metafilter.com/119873/O-soapy-flavour-Why-pollutest-thou-my-food-Thou-me-makest-retch">MetaFilter</a>
<a href="http://www.businessinsider.com/cilantro-corriander-taste-gene-found-2012-9">Business Insider</a>
<a href="http://www.huffingtonpost.com/2012/09/20/cilantro-aversion-gene-study_n_1901124.html">Huffington Post</a>
</p>
</div>
</div>
<hr>
</div>
<div>
<div class="grid">
<div class="g-col-2">
<h4 class="anchored">27</h4>
<img src="img/famhist.png" class="thumbnail">
</div>
<div class="g-col-1"></div>
<div class="g-col-9">
<p></p>
<p class="paper">
<strong>Comparison of Family History and SNPs for Predicting Risk of Complex Disease</strong>
</p>
<p class="authors paper">C. B. Do, D. A. Hinds, U. Francke, and N. Eriksson</p>
<p class="paper"><i>PLoS Genet.</i>, 8(10): e1002973, October 2012.</p>
<p class="paper">
<strong>Summary: </strong>
We develop models based on quantitative genetic theory to analyze and compare family history and SNP-based risk prediction models.
</p>
<p class="paper">
<strong>Links: </strong>
<a href="https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1002973">PLoS</a>
</p>
<p class="paper">
<strong>Press: </strong>
<a href="http://blog.sfgate.com/techchron/2012/10/17/genetics-firm-weaves-history-and-dna-for-predicting-diseases/">SF Chronicle</a>
</p>
</div>
</div>
<hr>
</div>
<div>
<div class="grid">
<div class="g-col-2">
<h4 class="anchored">26</h4>
<img src="img/bsize.png" class="thumbnail">
</div>
<div class="g-col-1"></div>
<div class="g-col-9">
<p></p>
<p class="paper">
<strong>Genetic variants associated with breast size also influence breast cancer risk</strong>
</p>
<p class="authors paper">N. Eriksson, G. M. Benton, C. B. Do, A. K. Kiefer, J. L. Mountain, D. A. Hinds, U. Francke, and J. Y. Tung</p>
<p class="paper"><i>BMC Med Genet</i>, 13(1):53, Jun 2012</p>
<p class="paper">
<strong>Summary: </strong>
We discover a number of SNPs associated with breast size. Surprisingly, several are also involved in breast cancer.
</p>
<p class="paper">
<strong>Links: </strong>
<a href="http://www.biomedcentral.com/1471-2350/13/53/abstract">BMC</a>
</p>
<p class="paper">
<strong>Press: </strong>
<a href="http://jezebel.com/5923807/scientists-find-the-genes-that-determine-breast-size">Jezebel</a>
<a href="http://www.huffingtonpost.com/2012/07/05/breast-cancer-risk-breast-size-study_n_1652292.html">Huffington Post</a>
<a href="http://www.telegraph.co.uk/health/healthnews/9377624/Breast-cancer-linked-to-bra-size.html">The Telegraph</a>
</p>
</div>
</div>
<hr>
</div>
<div>
<div class="grid">
<div class="g-col-2">
<h4 class="anchored">25</h4>
<img src="img/bald.png" class="thumbnail">
</div>
<div class="g-col-1"></div>
<div class="g-col-9">
<p></p>
<p class="paper">
<strong>Six novel susceptibility Loci for early-onset androgenetic alopecia and their unexpected association with common diseases</strong>
</p>
<p class="authors paper">R. Li, F. F. Brockschmidt, A. K. Kiefer, H. Stefansson, D. R. Nyholt, K. Song, S. H. Vermeulen, S. Kanoni, D. Glass, S. E. Medland, M. Dimitriou, D. Waterworth, J. Y. Tung, F. Geller, S. Heilmann, A. M. Hillmer, V. Bataille, S. Eigelshoven, S. Hanneken, S. Moebus, C. Herold, M. den Heijer, G. W. Montgomery, P. Deloukas, N. Eriksson, A. C. Heath, T. Becker, P. Sulem, M. Mangino, P. Vollenweider, T. D. Spector, G. Dedoussis, N. G.  Martin, L. A. Kiemeney, V. Mooser, K. Stefansson, D. A. Hinds, M. M. Nothen, and J. B. Richards</p>
<p class="paper"><i>PLoS Genet.</i>, 8(5):e1002746, May 2012.</p>
<p class="paper">
<strong>Summary: </strong>
A large meta-analysis of male-pattern baldness uncovers unexpected connections with Parkinson's disease
</p>
<p class="paper">
<strong>Links: </strong>
<a href="https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1002746">PLoS</a>
</p>
</div>
</div>
<hr>
</div>
<div>
<div class="grid">
<div class="g-col-2">
<h4 class="anchored">24</h4>
<img src="img/thyroid.png" class="thumbnail">
</div>
<div class="g-col-1"></div>
<div class="g-col-9">
<p></p>
<p class="paper">
<strong>Novel associations for hypothyroidism include known autoimmune risk loci</strong>
</p>
<p class="authors paper">N. Eriksson, J. Y. Tung, A. K. Kiefer, D. A. Hinds, U. Francke, J. L. Mountain, and C. B. Do</p>
<p class="paper"><i>PLoS ONE</i>, 7(4):e34442, 2012.</p>
<p class="paper">
<strong>Summary: </strong>
A GWAS for hypothyroidism implicates genes involved in both autoimmune disorders and thyroid function.
</p>
<p class="paper">
<strong>Links: </strong>
<a href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0034442">PLoS</a>
</p>
</div>
</div>
<hr>
</div>
<div>
<div class="grid">
<div class="g-col-2">
<h4 class="anchored">23</h4>
<img src="img/ibd.png" class="thumbnail">
</div>
<div class="g-col-1"></div>
<div class="g-col-9">
<p></p>
<p class="paper">
<strong>Cryptic distant relatives are common in both isolated and cosmopolitan genetic samples</strong>
</p>
<p class="authors paper">B. M. Henn, L. Hon, J. M. Macpherson, N. Eriksson, S. Saxonov, I. Pe'er, and J. L. Mountain</p>
<p class="paper"><i>PLoS ONE</i>, 7(4):e34267, 2012.</p>
<p class="paper">
<strong>Summary: </strong>
An analysis of the number of cryptic distant relatives among various populations.
</p>
<p class="paper">
<strong>Links: </strong>
<a href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0034267">PLoS</a>
</p>
</div>
</div>
<hr>
</div>
<div>
<div class="grid">
<div class="g-col-2">
<h4 class="anchored">22</h4>
<img src="img/replication.png" class="thumbnail">
</div>
<div class="g-col-1"></div>
<div class="g-col-9">
<p></p>
<p class="paper">
<strong>Efficient replication of over 180 genetic associations with self-reported medical data</strong>
</p>
<p class="authors paper">J. Y. Tung, C. B. Do, D. A. Hinds, A. K. Kiefer, J. M. Macpherson, A. B.  Chowdry, U. Francke, B. T. Naughton, J. L. Mountain, A. Wojcicki, and N. Eriksson</p>
<p class="paper"><i>PLoS ONE</i>, 6(8):e23473, 2011.</p>
<p class="paper">
<strong>Summary: </strong>
Replication of many associations using the 23andMe database.
</p>
<p class="paper">
<strong>Links: </strong>
<a href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0023473">PLoS</a>
</p>
</div>
</div>
<hr>
</div>
<div>
<div class="grid">
<div class="g-col-2">
<h4 class="anchored">21</h4>
<img src="img/pdgene.png" class="thumbnail">
</div>
<div class="g-col-1"></div>
<div class="g-col-9">
<p></p>
<p class="paper">
<strong>Comprehensive research synopsis and systematic meta-analyses in Parkinson's disease genetics: The PDGene database</strong>
</p>
<p class="authors paper">C. M. Lill, J. T. Roehr, M. B. McQueen, F. K. Kavvoura, S. Bagade, B. M.  Schjeide, L. M. Schjeide, E. Meissner, U. Zauft, N. C. Allen, T. Liu, M. Schilling, K. J. Anderson, G. Beecham, D. Berg, J. M. Biernacka, A. Brice, A. L. DeStefano, C. B. Do, N. Eriksson, S. A. Factor, M. J. Farrer, T. Foroud, T. Gasser, T. Hamza, J. A. Hardy, P. Heutink, E. M. Hill-Burns, C. Klein, J. C. Latourelle, D. M. Maraganore, E. R. Martin, M. Martinez, R. H. Myers, M. A. Nalls, N. Pankratz, H. Payami, W. Satake, W. K. Scott, M. Sharma, A. B. Singleton, K. Stefansson, T. Toda, J. Y. Tung, J. Vance, N. W. Wood, C. P. Zabetian, P. Young, R. E. Tanzi, M. J. Khoury, F. Zipp, H. Lehrach, J. P. Ioannidis, and L. Bertram</p>
<p class="paper"><i>PLoS Genet.</i>, 8(3):e1002548, 2012.</p>
<p class="paper">
<strong>Summary: </strong>
A comprehensive collection and meta-analysis of all genetic research in Parkinson's disease.
</p>
<p class="paper">
<strong>Links: </strong>
<a href="https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1002548">PLoS</a>
</p>
</div>
</div>
<hr>
</div>
<div>
<div class="grid">
<div class="g-col-2">
<h4 class="anchored">20</h4>
<img src="img/pathways.png" class="thumbnail">
</div>
<div class="g-col-1"></div>
<div class="g-col-9">
<p></p>
<p class="paper">
<strong>The temporal order of genetic and pathway alterations in tumorigenesis</strong>
</p>
<p class="authors paper">M. Gerstung, N. Eriksson, J. Lin, B. Vogelstein, and N. Beerenwinkel</p>
<p class="paper"><i>PLoS ONE</i>, 6(11):e27136, 2011.</p>
<p class="paper">
<strong>Summary: </strong>
Analysis of mutation accumulation in cancer using next-generation sequence data and conjunctive Bayesian networks.
</p>
<p class="paper">
<strong>Links: </strong>
<a href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0027136">PLoS</a>
</p>
</div>
</div>
<hr>
</div>
<div>
<div class="grid">
<div class="g-col-2">
<h4 class="anchored">19</h4>
<img src="img/pd.png" class="thumbnail">
</div>
<div class="g-col-1"></div>
<div class="g-col-9">
<p></p>
<p class="paper">
<strong>Web-based genome-wide association study identifies two novel loci and a substantial genetic component for Parkinson's disease</strong>
</p>
<p class="authors paper">C. B. Do, J. Y. Tung, E. Dorfman, A. K. Kiefer, E. M. Drabant, U. Francke, J. L. Mountain, S. M. Goldman, C. M. Tanner, J. W. Langston, A. Wojcicki, and N. Eriksson</p>
<p class="paper"><i>PLoS Genet.</i>, 7(6):e1002141, Jun 2011.</p>
<p class="paper">
<strong>Summary: </strong>
</p>
<p class="paper">
<strong>Links: </strong>
<a href="https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1002141">PLoS</a>
</p>
<p class="paper">
<strong>Press: </strong>
<a href="http://www.wired.com/magazine/2010/06/ff_sergeys_search/">Wired</a>
<a href="http://www.fastcoexist.com/1678218/23andme-moves-into-serious-genomic-research">FastCompany</a>
<a href="http://www.webmd.com/parkinsons-disease/news/20110624/new-genetic-clues-to-cause-of-parkinsons">WebMD</a>
<a href="http://www.nature.com/nature/journal/v475/n7357/full/475427f.html">Nature</a>
<a href="http://www.bloomberg.com/news/2012-05-11/google-s-brin-makes-strides-in-hunt-for-parkinson-s-cure-health.html">Bloomberg</a>
</p>
</div>
</div>
<hr>
</div>
<div>
<div class="grid">
<div class="g-col-2">
<h4 class="anchored">18</h4>
<img src="img/parametric.png" class="thumbnail">
</div>
<div class="g-col-1"></div>
<div class="g-col-9">
<p></p>
<p class="paper">
<strong>Parametric analysis of alignment and phylogenetic uncertainty</strong>
</p>
<p class="authors paper">A. S. Malaspinas, N. Eriksson, and P. Huggins</p>
<p class="paper"><i>Bull. Math. Biol.</i>, 73:795--810, Apr 2011.</p>
<p class="paper">
<strong>Summary: </strong>
We show how to analyze the robustness of phylogenies to perturbations in alignment parameters.
</p>
<p class="paper">
<strong>Links: </strong>
<a href="http://link.springer.com/article/10.1007%2Fs11538-010-9610-8">BMB</a>
</p>
</div>
</div>
<hr>
</div>
<div>
<div class="grid">
<div class="g-col-2">
<h4 class="anchored">17</h4>
<img src="img/shorah.png" class="thumbnail">
</div>
<div class="g-col-1"></div>
<div class="g-col-9">
<p></p>
<p class="paper">
<strong>ShoRAH: estimating the genetic diversity of a mixed sample from next-generation sequencing data</strong>
</p>
<p class="authors paper">O. Zagordi, A. Bhattacharya, N. Eriksson, and N. Beerenwinkel</p>
<p class="paper"><i>BMC Bioinformatics</i>, 12:119, Apr 2011.</p>
<p class="paper">
<strong>Summary: </strong>
</p>
<p class="paper">
<strong>Links: </strong>
<a href="http://www.biomedcentral.com/1471-2105/12/119">BMC</a>
<a href="http://www.silva.bsse.ethz.ch/cbg/software/shorah">Software</a>
</p>
</div>
</div>
<hr>
</div>
<div>
<div class="grid">
<div class="g-col-2">
<h4 class="anchored">16</h4>
<img src="img/common.png" class="thumbnail">
</div>
<div class="g-col-1"></div>
<div class="g-col-9">
<p></p>
<p class="paper">
<strong>Web-based, participant-driven studies yield novel genetic associations for common traits</strong>
</p>
<p class="authors paper">N. Eriksson, J. M. Macpherson, J. Y. Tung, L. S. Hon, B. Naughton, S. Saxonov, L. Avey, A. Wojcicki, I. Pe'er, and J. Mountain</p>
<p class="paper"><i>PLoS Genet.</i>, 6:e1000993, Jun 2010.</p>
<p class="paper">
<strong>Summary: </strong>
We discover novel genetic associations for hair curl, photic sneeze reflex, freckling, and asparagus metabolite detection.
</p>
<p class="paper">
<strong>Links: </strong>
<a href="https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1000993">PLoS</a>
</p>
<p class="paper">
<strong>Press: </strong>
<a href="http://f1000.com/4871956">F1000</a>
<a href="http://blogs.nature.com/spoonful/2010/06/gene_found_for_smelling_aspara.html">Nature</a>
<a href="http://abcnews.go.com/Health/Wellness/direct-consumer-genetic-tests-research-bonanza/story?id=11016734#.UHuYYWl25kA">ABC</a>
</p>
</div>
</div>
<hr>
</div>
<div>
<div class="grid">
<div class="g-col-2">
<h4 class="anchored">15</h4>
<img src="img/using.png" class="thumbnail">
</div>
<div class="g-col-1"></div>
<div class="g-col-9">
<p></p>
<p class="paper">
<strong>Using invariants for phylogenetic tree construction</strong>
</p>
<p class="authors paper">N. Eriksson</p>
<p class="paper">In <i>Emerging Applications of Algebraic Geometry</i>, pages 89--108. Springer, New York, 2009.</p>
<p class="paper">
<strong>Summary: </strong>
A survey of how to use algebraic techniques to solve practical problems in phylogenetics.
</p>
<p class="paper">
<strong>Links: </strong>
<a href="http://arxiv.org/abs/0709.2890">arXiv</a>
<a href="pdf/using.pdf">pdf</a>
</p>
</div>
</div>
<hr>
</div>
<div>
<div class="grid">
<div class="g-col-2">
<h4 class="anchored">14</h4>
<img src="img/viral.png" class="thumbnail">
</div>
<div class="g-col-1"></div>
<div class="g-col-9">
<p></p>
<p class="paper">
<strong>Viral population estimation using pyrosequencing</strong>
</p>
<p class="authors paper">N. Eriksson, L. Pachter, Y. Mitsuya, S. Y. Rhee, C. Wang, B. Gharizadeh, M. Ronaghi, R. W. Shafer, and N. Beerenwinkel</p>
<p class="paper"><i>PLoS Comput. Biol.</i>, 4:e1000074, Apr 2008.</p>
<p class="paper">
<strong>Summary: </strong>
We show how to reconstruct an entire viral population from next-generation sequencing data.
</p>
<p class="paper">
<strong>Links: </strong>
<a href="http://www.ploscompbiol.org/doi/pcbi.1000074">PLoS</a>
</p>
</div>
</div>
<hr>
</div>
<div>
<div class="grid">
<div class="g-col-2">
<h4 class="anchored">13</h4>
<img src="img/apobec.png" class="thumbnail">
</div>
<div class="g-col-1"></div>
<div class="g-col-9">
<p></p>
<p class="paper">
<strong>Sequence editing by Apolipoprotein B RNA-editing catalytic component and epidemiological surveillance of transmitted HIV-1 drug resistance</strong>
</p>
<p class="authors paper">R. J. Gifford, S. Y. Rhee, N. Eriksson, T. F. Liu, M. Kiuchi, A. K. Das, and R. W. Shafer</p>
<p class="paper"><i>AIDS</i>, 22:717--725, Mar 2008.</p>
<p class="paper">
<strong>Summary: </strong>
We examine APOBEC-mediated editing of HIV-1 and show that this editing is meaningful for the study of drug resistance.
</p>
<p class="paper">
<strong>Links: </strong>
<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2946849/">PubMed Central</a>
<a href="https://journals.lww.com/aidsonline/abstract/2008/03300/sequence_editing_by_apolipoprotein_b_rna_editing.6.aspx">AIDS</a>
</p>
</div>
</div>
<hr>
</div>
<div>
<div class="grid">
<div class="g-col-2">
<h4 class="anchored">12</h4>
<img src="img/cbn.png" class="thumbnail">
</div>
<div class="g-col-1"></div>
<div class="g-col-9">
<p></p>
<p class="paper">
<strong>Conjuctive Bayesian networks</strong>
</p>
<p class="authors paper">N. Beerenwinkel, N. Eriksson, and B. Sturmfels</p>
<p class="paper"><i>Bernoulli</i>, 13(4):893--909, 2007.</p>
<p class="paper">
<strong>Summary: </strong>
A new statistical model for describing the accumulation of events that are constrained in the order of their occurrence, applied to HIV drug resistance.
</p>
<p class="paper">
<strong>Links: </strong>
<a href="http://dx.doi.org/10.3150/07-BEJ6133">DOI</a>
<a href="pdf/CBN.pdf">pdf</a>
</p>
</div>
</div>
<hr>
</div>
<div>
<div class="grid">
<div class="g-col-2">
<h4 class="anchored">11</h4>
<img src="img/acp.png" class="thumbnail">
</div>
<div class="g-col-1"></div>
<div class="g-col-9">
<p></p>
<p class="paper">
<strong>Apollonian Circle Packings: Number Theory II. Spherical and Hyperbolic Packings</strong>
</p>
<p class="authors paper">N. Eriksson and J. C. Lagarias</p>
<p class="paper"><i>Ramanujan Journal</i>, 14(3):437--469, 2007.</p>
<p class="paper">
<strong>Summary: </strong>
A study of Apollonian circule packings in spherical and hyperbolic geometries
</p>
<p class="paper">
<strong>Links: </strong>
<a href="http://dx.doi.org/10.1007/s11139-007-9052-6">DOI</a>
<a href="pdf/acp.pdf">pdf</a>
</p>
</div>
</div>
<hr>
</div>
<div>
<div class="grid">
<div class="g-col-2">
<h4 class="anchored">10</h4>
<img src="img/metric.png" class="thumbnail">
</div>
<div class="g-col-1"></div>
<div class="g-col-9">
<p></p>
<p class="paper">
<strong>Metric learning for phylogenetic invariants</strong>
</p>
<p class="authors paper">N. Eriksson and Y. Yao</p>
<p class="paper"><i>ArXiv preprint</i>, 2007.</p>
<p class="paper">
<strong>Summary: </strong>
A novel use of machine learning to optimize the use of algebraic techniques in phylogenetics.
</p>
<p class="paper">
<strong>Links: </strong>
<a href="http://arxiv.org/abs/q-bio.PE/0703034">arXiv</a>
</p>
</div>
</div>
<hr>
</div>
<div>
<div class="grid">
<div class="g-col-2">
<h4 class="anchored">9</h4>
<img src="img/markov.png" class="thumbnail">
</div>
<div class="g-col-1"></div>
<div class="g-col-9">
<p></p>
<p class="paper">
<strong>Markov bases for noncommutative Fourier analysis of ranked data</strong>
</p>
<p class="authors paper">P. Diaconis and N. Eriksson</p>
<p class="paper"><i>J. Symbolic Comput.</i>, 41(2):182--195, 2006.</p>
<p class="paper">
<strong>Summary: </strong>
A combination of statistics, commutative algebra, and the representation theory of the symmetric group.
</p>
<p class="paper">
<strong>Links: </strong>
<a href="http://dx.doi.org/10.1016/j.jsc.2005.04.009">DOI</a>
</p>
</div>
</div>
<hr>
</div>
<div>
<div class="grid">
<div class="g-col-2">
<h4 class="anchored">8</h4>
<img src="img/mle.png" class="thumbnail">
</div>
<div class="g-col-1"></div>
<div class="g-col-9">
<p></p>
<p class="paper">
<strong>Polyhedral conditions for the nonexistence of the MLE for hierarchical log-linear models</strong>
</p>
<p class="authors paper">N. Eriksson, S. E. Fienberg, A. Rinaldo, and S. Sullivant</p>
<p class="paper"><i>J. Symbolic Comput.</i>, 41(2):222--233, 2006.</p>
<p class="paper">
<strong>Summary: </strong>
We give geometric conditions for the existence of the maximum likelihood estimate (MLE) for a hierarchical log-linear model.
</p>
<p class="paper">
<strong>Links: </strong>
<a href="http://dx.doi.org/10.1016/j.jsc.2005.04.003">DOI</a>
</p>
</div>
</div>
<hr>
</div>
<div>
<div class="grid">
<div class="g-col-2">
<h4 class="anchored">7</h4>
<img src="img/evoldl.png" class="thumbnail">
</div>
<div class="g-col-1"></div>
<div class="g-col-9">
<p></p>
<p class="paper">
<strong>Evolution on distributive lattices</strong>
</p>
<p class="authors paper">N. Beerenwinkel, N. Eriksson, and B. Sturmfels</p>
<p class="paper"><i>J Theor Biol</i>, 242(2):409--420, Sep 2006.</p>
<p class="paper">
<strong>Summary: </strong>
We develop a new mathematical model in order to study the development of HIV drug resistance.
</p>
<p class="paper">
<strong>Links: </strong>
<a href="pdf/evol.pdf">pdf</a>
</p>
</div>
</div>
<hr>
</div>
<div>
<div class="grid">
<div class="g-col-2">
<h4 class="anchored">6</h4>
<img src="img/thesis.png" class="thumbnail">
</div>
<div class="g-col-1"></div>
<div class="g-col-9">
<p></p>
<p class="paper">
<strong>Algebraic combinatorics for computational biology</strong>
</p>
<p class="authors paper">N. Eriksson</p>
<p class="paper">PhD thesis, University of California, Berkeley, 2006.</p>
<p class="paper">
<strong>Summary: </strong>
My PhD thesis.
</p>
<p class="paper">
<strong>Links: </strong>
<a href="pdf/nke-diss.pdf">pdf</a>
</p>
</div>
</div>
<hr>
</div>
<div>
<div class="grid">
<div class="g-col-2">
<h4 class="anchored">5</h4>
<img src="img/phyloag.png" class="thumbnail">
</div>
<div class="g-col-1"></div>
<div class="g-col-9">
<p></p>
<p class="paper">
<strong>Phylogenetic algebraic geometry</strong>
</p>
<p class="authors paper">N. Eriksson,  K. Ranestad, B. Sturmfels, and S. Sullivant</p>
<p class="paper">In C. Ciliberto, A. Geramita, B. Harbourne, R-M. Roig, and K. Ranestad, editors, <i>Projective varieties with unexpected properties</i>, pages 237--255. Walter de Gruyter GmbH &amp; Co. KG, Berlin, 2005.</p>
<p class="paper">
<strong>Summary: </strong>
An introduction to phylogenetics for algebraic geometers.
</p>
<p class="paper">
<strong>Links: </strong>
<a href="pdf/phylogeneticAG.pdf">pdf</a>
</p>
</div>
</div>
<hr>
</div>
<div>
<div class="grid">
<div class="g-col-2">
<h4 class="anchored">4</h4>
<img src="img/ultra.png" class="thumbnail">
</div>
<div class="g-col-1"></div>
<div class="g-col-9">
<p></p>
<p class="paper">
<strong>Ultra-Conserved Elements in Vertebrate and Fly Genomes</strong>
</p>
<p class="authors paper">M. Drton, N. Eriksson, and G. Leung</p>
<p class="paper">In L. Pachter and B. Sturmfels, editors, <i>Algebraic Statistics for Computational Biology</i>, chapter 22, pages 387--402. Cambridge University Press, Cambridge, UK, 2005.</p>
<p class="paper">
<strong>Summary: </strong>
An analysis of some of the most conserved DNA sequences
</p>
<p class="paper">
<strong>Links: </strong>
<a href="pdf/ultras.pdf">pdf</a>
</p>
</div>
</div>
<hr>
</div>
<div>
<div class="grid">
<div class="g-col-2">
<h4 class="anchored">3</h4>
<img src="img/treesvd.png" class="thumbnail">
</div>
<div class="g-col-1"></div>
<div class="g-col-9">
<p></p>
<p class="paper">
<strong>Tree Construction using Singular Value Decompsition</strong>
</p>
<p class="authors paper">N. Eriksson</p>
<p class="paper">In L. Pachter and B. Sturmfels, editors, <i>Algebraic Statistics for Computational Biology</i>, chapter 19, pages 347--358. Cambridge University Press, Cambridge, UK, 2005.</p>
<p class="paper">
<strong>Summary: </strong>
A novel technique for constructing phylogenies using linear algebra.
</p>
<p class="paper">
<strong>Links: </strong>
<a href="pdf/treeSVD.pdf">pdf</a>
</p>
</div>
</div>
<hr>
</div>
<div>
<div class="grid">
<div class="g-col-2">
<h4 class="anchored">2</h4>
<img src="img/toric2.png" class="thumbnail">
</div>
<div class="g-col-1"></div>
<div class="g-col-9">
<p></p>
<p class="paper">
<strong>Toric ideals of homogeneous phylogenetic models</strong>
</p>
<p class="authors paper">N. Eriksson</p>
<p class="paper">In the proceedings of <i>ISSAC 2004</i>, pages 149--154. ACM, New York, 2004.</p>
<p class="paper">
<strong>Summary: </strong>
An application of some algebraic techniques to a simple problem in phylogenetics.
</p>
<p class="paper">
<strong>Links: </strong>
<a href="pdf/phylogenetic-paper.pdf">pdf</a>
<a href="http://dl.acm.org/citation.cfm?doid=1005285.1005308">DOI</a>
</p>
</div>
</div>
<hr>
</div>
<div>
<div class="grid">
<div class="g-col-2">
<h4 class="anchored">1</h4>
<img src="img/partitions.png" class="thumbnail">
</div>
<div class="g-col-1"></div>
<div class="g-col-9">
<p></p>
<p class="paper">
<strong>$q$-series, elliptic curves, and odd values of the partition function</strong>
</p>
<p class="authors paper">N. Eriksson</p>
<p class="paper"><i>International Journal of Mathematics and Mathematical Sciences</i>, 22(1):55--66, 1999</p>
<p class="paper">
<strong>Summary: </strong>
A number theory paper that uses a bit of technical machinery to find odd values of the partition function. A high-school project that won third place in the Westinghouse Science Talent Search.
</p>
<p class="paper">
<strong>Links: </strong>
<a href="pdf/q-series_elliptic_curves_partition_function.pdf">pdf</a>
</p>
<p class="paper">
<strong>Press: </strong>
<a href="http://www.maa.org/mathland/mathland_3_24.html">Mathland</a>
</p>
</div>
</div>
<hr>
</div>
</div>
</div>
</div>



</main> <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
  window.document.addEventListener("DOMContentLoaded", function (event) {
    const icon = "";
    const anchorJS = new window.AnchorJS();
    anchorJS.options = {
      placement: 'right',
      icon: icon
    };
    anchorJS.add('.anchored');
    const isCodeAnnotation = (el) => {
      for (const clz of el.classList) {
        if (clz.startsWith('code-annotation-')) {                     
          return true;
        }
      }
      return false;
    }
    const onCopySuccess = function(e) {
      // button target
      const button = e.trigger;
      // don't keep focus
      button.blur();
      // flash "checked"
      button.classList.add('code-copy-button-checked');
      var currentTitle = button.getAttribute("title");
      button.setAttribute("title", "Copied!");
      let tooltip;
      if (window.bootstrap) {
        button.setAttribute("data-bs-toggle", "tooltip");
        button.setAttribute("data-bs-placement", "left");
        button.setAttribute("data-bs-title", "Copied!");
        tooltip = new bootstrap.Tooltip(button, 
          { trigger: "manual", 
            customClass: "code-copy-button-tooltip",
            offset: [0, -8]});
        tooltip.show();    
      }
      setTimeout(function() {
        if (tooltip) {
          tooltip.hide();
          button.removeAttribute("data-bs-title");
          button.removeAttribute("data-bs-toggle");
          button.removeAttribute("data-bs-placement");
        }
        button.setAttribute("title", currentTitle);
        button.classList.remove('code-copy-button-checked');
      }, 1000);
      // clear code selection
      e.clearSelection();
    }
    const getTextToCopy = function(trigger) {
      const outerScaffold = trigger.parentElement.cloneNode(true);
      const codeEl = outerScaffold.querySelector('code');
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
    }
    const clipboard = new window.ClipboardJS('.code-copy-button:not([data-in-quarto-modal])', {
      text: getTextToCopy
    });
    clipboard.on('success', onCopySuccess);
    if (window.document.getElementById('quarto-embedded-source-code-modal')) {
      const clipboardModal = new window.ClipboardJS('.code-copy-button[data-in-quarto-modal]', {
        text: getTextToCopy,
        container: window.document.getElementById('quarto-embedded-source-code-modal')
      });
      clipboardModal.on('success', onCopySuccess);
    }
      var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
      var mailtoRegex = new RegExp(/^mailto:/);
        var filterRegex = new RegExp('/' + window.location.host + '/');
      var isInternal = (href) => {
          return filterRegex.test(href) || localhostRegex.test(href) || mailtoRegex.test(href);
      }
      // Inspect non-navigation links and adorn them if external
     var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item):not(.quarto-navigation-tool):not(.about-link)');
      for (var i=0; i<links.length; i++) {
        const link = links[i];
        if (!isInternal(link.href)) {
          // undo the damage that might have been done by quarto-nav.js in the case of
          // links that we want to consider external
          if (link.dataset.originalHref !== undefined) {
            link.href = link.dataset.originalHref;
          }
        }
      }
    function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
      const config = {
        allowHTML: true,
        maxWidth: 500,
        delay: 100,
        arrow: false,
        appendTo: function(el) {
            return el.parentElement;
        },
        interactive: true,
        interactiveBorder: 10,
        theme: 'quarto',
        placement: 'bottom-start',
      };
      if (contentFn) {
        config.content = contentFn;
      }
      if (onTriggerFn) {
        config.onTrigger = onTriggerFn;
      }
      if (onUntriggerFn) {
        config.onUntrigger = onUntriggerFn;
      }
      window.tippy(el, config); 
    }
    const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
    for (var i=0; i<noterefs.length; i++) {
      const ref = noterefs[i];
      tippyHover(ref, function() {
        // use id or data attribute instead here
        let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
        try { href = new URL(href).hash; } catch {}
        const id = href.replace(/^#\/?/, "");
        const note = window.document.getElementById(id);
        if (note) {
          return note.innerHTML;
        } else {
          return "";
        }
      });
    }
    const xrefs = window.document.querySelectorAll('a.quarto-xref');
    const processXRef = (id, note) => {
      // Strip column container classes
      const stripColumnClz = (el) => {
        el.classList.remove("page-full", "page-columns");
        if (el.children) {
          for (const child of el.children) {
            stripColumnClz(child);
          }
        }
      }
      stripColumnClz(note)
      if (id === null || id.startsWith('sec-')) {
        // Special case sections, only their first couple elements
        const container = document.createElement("div");
        if (note.children && note.children.length > 2) {
          container.appendChild(note.children[0].cloneNode(true));
          for (let i = 1; i < note.children.length; i++) {
            const child = note.children[i];
            if (child.tagName === "P" && child.innerText === "") {
              continue;
            } else {
              container.appendChild(child.cloneNode(true));
              break;
            }
          }
          if (window.Quarto?.typesetMath) {
            window.Quarto.typesetMath(container);
          }
          return container.innerHTML
        } else {
          if (window.Quarto?.typesetMath) {
            window.Quarto.typesetMath(note);
          }
          return note.innerHTML;
        }
      } else {
        // Remove any anchor links if they are present
        const anchorLink = note.querySelector('a.anchorjs-link');
        if (anchorLink) {
          anchorLink.remove();
        }
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(note);
        }
        if (note.classList.contains("callout")) {
          return note.outerHTML;
        } else {
          return note.innerHTML;
        }
      }
    }
    for (var i=0; i<xrefs.length; i++) {
      const xref = xrefs[i];
      tippyHover(xref, undefined, function(instance) {
        instance.disable();
        let url = xref.getAttribute('href');
        let hash = undefined; 
        if (url.startsWith('#')) {
          hash = url;
        } else {
          try { hash = new URL(url).hash; } catch {}
        }
        if (hash) {
          const id = hash.replace(/^#\/?/, "");
          const note = window.document.getElementById(id);
          if (note !== null) {
            try {
              const html = processXRef(id, note.cloneNode(true));
              instance.setContent(html);
            } finally {
              instance.enable();
              instance.show();
            }
          } else {
            // See if we can fetch this
            fetch(url.split('#')[0])
            .then(res => res.text())
            .then(html => {
              const parser = new DOMParser();
              const htmlDoc = parser.parseFromString(html, "text/html");
              const note = htmlDoc.getElementById(id);
              if (note !== null) {
                const html = processXRef(id, note);
                instance.setContent(html);
              } 
            }).finally(() => {
              instance.enable();
              instance.show();
            });
          }
        } else {
          // See if we can fetch a full url (with no hash to target)
          // This is a special case and we should probably do some content thinning / targeting
          fetch(url)
          .then(res => res.text())
          .then(html => {
            const parser = new DOMParser();
            const htmlDoc = parser.parseFromString(html, "text/html");
            const note = htmlDoc.querySelector('main.content');
            if (note !== null) {
              // This should only happen for chapter cross references
              // (since there is no id in the URL)
              // remove the first header
              if (note.children.length > 0 && note.children[0].tagName === "HEADER") {
                note.children[0].remove();
              }
              const html = processXRef(null, note);
              instance.setContent(html);
            } 
          }).finally(() => {
            instance.enable();
            instance.show();
          });
        }
      }, function(instance) {
      });
    }
        let selectedAnnoteEl;
        const selectorForAnnotation = ( cell, annotation) => {
          let cellAttr = 'data-code-cell="' + cell + '"';
          let lineAttr = 'data-code-annotation="' +  annotation + '"';
          const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
          return selector;
        }
        const selectCodeLines = (annoteEl) => {
          const doc = window.document;
          const targetCell = annoteEl.getAttribute("data-target-cell");
          const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
          const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
          const lines = annoteSpan.getAttribute("data-code-lines").split(",");
          const lineIds = lines.map((line) => {
            return targetCell + "-" + line;
          })
          let top = null;
          let height = null;
          let parent = null;
          if (lineIds.length > 0) {
              //compute the position of the single el (top and bottom and make a div)
              const el = window.document.getElementById(lineIds[0]);
              top = el.offsetTop;
              height = el.offsetHeight;
              parent = el.parentElement.parentElement;
            if (lineIds.length > 1) {
              const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
              const bottom = lastEl.offsetTop + lastEl.offsetHeight;
              height = bottom - top;
            }
            if (top !== null && height !== null && parent !== null) {
              // cook up a div (if necessary) and position it 
              let div = window.document.getElementById("code-annotation-line-highlight");
              if (div === null) {
                div = window.document.createElement("div");
                div.setAttribute("id", "code-annotation-line-highlight");
                div.style.position = 'absolute';
                parent.appendChild(div);
              }
              div.style.top = top - 2 + "px";
              div.style.height = height + 4 + "px";
              div.style.left = 0;
              let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
              if (gutterDiv === null) {
                gutterDiv = window.document.createElement("div");
                gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
                gutterDiv.style.position = 'absolute';
                const codeCell = window.document.getElementById(targetCell);
                const gutter = codeCell.querySelector('.code-annotation-gutter');
                gutter.appendChild(gutterDiv);
              }
              gutterDiv.style.top = top - 2 + "px";
              gutterDiv.style.height = height + 4 + "px";
            }
            selectedAnnoteEl = annoteEl;
          }
        };
        const unselectCodeLines = () => {
          const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
          elementsIds.forEach((elId) => {
            const div = window.document.getElementById(elId);
            if (div) {
              div.remove();
            }
          });
          selectedAnnoteEl = undefined;
        };
          // Handle positioning of the toggle
      window.addEventListener(
        "resize",
        throttle(() => {
          elRect = undefined;
          if (selectedAnnoteEl) {
            selectCodeLines(selectedAnnoteEl);
          }
        }, 10)
      );
      function throttle(fn, ms) {
      let throttle = false;
      let timer;
        return (...args) => {
          if(!throttle) { // first call gets through
              fn.apply(this, args);
              throttle = true;
          } else { // all the others get throttled
              if(timer) clearTimeout(timer); // cancel #2
              timer = setTimeout(() => {
                fn.apply(this, args);
                timer = throttle = false;
              }, ms);
          }
        };
      }
        // Attach click handler to the DT
        const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
        for (const annoteDlNode of annoteDls) {
          annoteDlNode.addEventListener('click', (event) => {
            const clickedEl = event.target;
            if (clickedEl !== selectedAnnoteEl) {
              unselectCodeLines();
              const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
              if (activeEl) {
                activeEl.classList.remove('code-annotation-active');
              }
              selectCodeLines(clickedEl);
              clickedEl.classList.add('code-annotation-active');
            } else {
              // Unselect the line
              unselectCodeLines();
              clickedEl.classList.remove('code-annotation-active');
            }
          });
        }
    const findCites = (el) => {
      const parentEl = el.parentElement;
      if (parentEl) {
        const cites = parentEl.dataset.cites;
        if (cites) {
          return {
            el,
            cites: cites.split(' ')
          };
        } else {
          return findCites(el.parentElement)
        }
      } else {
        return undefined;
      }
    };
    var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
    for (var i=0; i<bibliorefs.length; i++) {
      const ref = bibliorefs[i];
      const citeInfo = findCites(ref);
      if (citeInfo) {
        tippyHover(citeInfo.el, function() {
          var popup = window.document.createElement('div');
          citeInfo.cites.forEach(function(cite) {
            var citeDiv = window.document.createElement('div');
            citeDiv.classList.add('hanging-indent');
            citeDiv.classList.add('csl-entry');
            var biblioDiv = window.document.getElementById('ref-' + cite);
            if (biblioDiv) {
              citeDiv.innerHTML = biblioDiv.innerHTML;
            }
            popup.appendChild(citeDiv);
          });
          return popup.innerHTML;
        });
      }
    }
  });
  </script>
</div> <!-- /content -->




</body></html>